diff -Naur frontline_battle.orig-0.orig/net/wg/data/constants/Linkages.class.asasm frontline_battle.orig-0/net/wg/data/constants/Linkages.class.asasm
--- frontline_battle.orig-0.orig/net/wg/data/constants/Linkages.class.asasm     2025-08-23 03:04:41.649843800 +0300
+++ frontline_battle.orig-0/net/wg/data/constants/Linkages.class.asasm  2025-08-23 03:15:52.663902300 +0300
@@ -2401,7 +2401,7 @@
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_ITEM_LABEL") slotid 292 type QName(PackageNamespace(""), "String") value Utf8("battleProgressItemLabelUI") end
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_ITEM_LINE") slotid 293 type QName(PackageNamespace(""), "String") value Utf8("battleProgressItemLineUI") end
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_TASK_SEPARATOR") slotid 294 type QName(PackageNamespace(""), "String") value Utf8("battleProgressTaskSeparatorUI") end
- trait const QName(PackageNamespace(""), "CAPTURE_BAR_LINKAGE") slotid 295 type QName(PackageNamespace(""), "String") value Utf8("TeamCaptureBarUI") end
+ trait slot QName(PackageNamespace(""), "CAPTURE_BAR_LINKAGE") slotid 295 type QName(PackageNamespace(""), "String") value Utf8("TeamCaptureBarUI") end
  trait const QName(PackageNamespace(""), "VEH_CMP_WINDOW_BG_UI") slotid 296 type QName(PackageNamespace(""), "String") value Utf8("WindowBgUI") end
  trait const QName(PackageNamespace(""), "VEH_CMP_BUBBLE_UI") slotid 297 type QName(PackageNamespace(""), "String") value Utf8("VehCompareBubbleUI") end
  trait const QName(PackageNamespace(""), "DIP_ENGINE_DAMAGED") slotid 298 type QName(PackageNamespace(""), "String") value Utf8("DipEngineDamagedUI") end
