diff -Naur lobby.orig-0.orig/net/wg/gui/messenger/data/ContactItemVO.class.asasm lobby.orig-0/net/wg/gui/messenger/data/ContactItemVO.class.asasm
--- lobby.orig-0.orig/net/wg/gui/messenger/data/ContactItemVO.class.asasm	2016-11-18 23:06:09.386172500 +0200
+++ lobby.orig-0/net/wg/gui/messenger/data/ContactItemVO.class.asasm	2016-11-18 23:07:06.536607100 +0200
@@ -25,6 +25,7 @@
     end ; code
    end ; body
   end ; method
+  trait slot QName(PackageNamespace(""), "xvm_contact_data") type QName(PackageNamespace(""), "Object") value Null() end
   trait slot QName(PackageNamespace(""), "dbID") type QName(PackageNamespace(""), "Number") value Integer(0) end
   trait slot QName(PackageNamespace(""), "note") type QName(PackageNamespace(""), "String") value Utf8("") end
   trait slot QName(PackageNamespace(""), "resource") type QName(PackageNamespace(""), "String") value Utf8("") end
