diff -Naur battle.orig-0.orig/net/wg/data/constants/Linkages.class.asasm battle.orig-0/net/wg/data/constants/Linkages.class.asasm
--- battle.orig-0.orig/net/wg/data/constants/Linkages.class.asasm       2024-08-27 00:25:27.678286400 +0000
+++ battle.orig-0/net/wg/data/constants/Linkages.class.asasm    2024-08-27 00:31:49.815590400 +0000
@@ -2461,7 +2461,7 @@
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_ITEM_LABEL") slotid 280 type QName(PackageNamespace(""), "String") value Utf8("battleProgressItemLabelUI") end
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_ITEM_LINE") slotid 281 type QName(PackageNamespace(""), "String") value Utf8("battleProgressItemLineUI") end
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_TASK_SEPARATOR") slotid 282 type QName(PackageNamespace(""), "String") value Utf8("battleProgressTaskSeparatorUI") end
- trait const QName(PackageNamespace(""), "CAPTURE_BAR_LINKAGE") slotid 283 type QName(PackageNamespace(""), "String") value Utf8("TeamCaptureBarUI") end
+ trait slot QName(PackageNamespace(""), "CAPTURE_BAR_LINKAGE") slotid 283 type QName(PackageNamespace(""), "String") value Utf8("TeamCaptureBarUI") end
  trait const QName(PackageNamespace(""), "VEH_CMP_WINDOW_BG_UI") slotid 284 type QName(PackageNamespace(""), "String") value Utf8("WindowBgUI") end
  trait const QName(PackageNamespace(""), "VEH_CMP_BUBBLE_UI") slotid 285 type QName(PackageNamespace(""), "String") value Utf8("VehCompareBubbleUI") end
  trait const QName(PackageNamespace(""), "DIP_ENGINE_DAMAGED") slotid 286 type QName(PackageNamespace(""), "String") value Utf8("DipEngineDamagedUI") end
