diff -Naur battleVehicleMarkersApp.orig-0.orig/net/wg/gui/battle/views/vehicleMarkers/VehicleMarkersManager.class.asasm battleVehicleMarkersApp.orig-0/net/wg/gui/battle/views/vehicleMarkers/VehicleMarkersManager.class.asasm
--- battleVehicleMarkersApp.orig-0.orig/net/wg/gui/battle/views/vehicleMarkers/VehicleMarkersManager.class.asasm	2022-10-09 21:37:07.393239700 +0300
+++ battleVehicleMarkersApp.orig-0/net/wg/gui/battle/views/vehicleMarkers/VehicleMarkersManager.class.asasm	2022-10-09 21:46:27.436818900 +0300
@@ -843,6 +843,29 @@
     end ; body
    end ; method
   end ; trait
+  trait method QName(PackageNamespace(""), "daapi_py2as")
+   method
+    flag NEED_REST
+    body
+     maxstack 3
+     localcount 2
+     initscopedepth 14
+     maxscopedepth 15
+     code
+      getlocal0
+      pushscope
+
+      debug               1, "rest", 0, 0
+      getlex              QName(PackageNamespace("com.xfw"), "Xfw")
+      getproperty         QName(PackageNamespace(""), "as_xfw_cmd")
+      getlocal0
+      getlocal1
+      callproperty        QName(Namespace("http://adobe.com/AS3/2006/builtin"), "apply"), 2
+      returnvalue
+     end ; code
+    end ; body
+   end ; method
+  end ; trait
  end ; instance
  cinit
   refid "net.wg.gui.battle.views.vehicleMarkers:VehicleMarkersManager/class/init"
