diff -Naur battleVehicleMarkersApp.orig-0.orig/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.class.asasm battleVehicleMarkersApp.orig-0/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.class.asasm
--- battleVehicleMarkersApp.orig-0.orig/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.class.asasm	2022-05-28 14:20:43.836763500 +0300
+++ battleVehicleMarkersApp.orig-0/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.class.asasm	2022-05-28 14:23:28.967011700 +0300
@@ -2029,6 +2029,24 @@
     end ; body
    end ; method
   end ; trait
+  trait method QName(PackageNamespace(""), "xvm_active")
+   method
+    returns QName(PackageNamespace(""), "Boolean")
+    body
+     maxstack 1
+     localcount 1
+     initscopedepth 11
+     maxscopedepth 12
+     code
+      getlocal0
+      pushscope
+
+      pushfalse
+      returnvalue
+     end ; code
+    end ; body
+   end ; method
+  end ; trait
   trait method QName(PrivateNamespace(null, "net.wg.gui.battle.views.vehicleMarkers:VehicleMarker"), "updateMarkerSettings")
    method
     refid "net.wg.gui.battle.views.vehicleMarkers:VehicleMarker/instance/net.wg.gui.battle.views.vehicleMarkers:VehicleMarker/updateMarkerSettings"
@@ -2047,6 +2065,13 @@
       setlocal            9
 
       getlocal0
+      callproperty        QName(PackageNamespace(""), "xvm_active"), 0
+      iffalse             XVMACTIVE
+
+      returnvoid
+
+XVMACTIVE:
+      getlocal0
       getlex              QName(PrivateNamespace(null, "net.wg.gui.battle.views.vehicleMarkers:VehicleMarker"), "ICON")
       callproperty        QName(PrivateNamespace(null, "net.wg.gui.battle.views.vehicleMarkers:VehicleMarker"), "getIsPartVisible"), 1
       convert_b
@@ -4076,6 +4101,26 @@
     end ; body
    end ; method
   end ; trait
+  trait method QName(PackageNamespace(""), "xfw_updateIconColor")
+   method
+    returns QName(PackageNamespace(""), "void")
+    body
+     maxstack 1
+     localcount 1
+     initscopedepth 15
+     maxscopedepth 16
+     code
+      getlocal0
+      pushscope
+
+      getlocal0
+      callpropvoid        QName(PrivateNamespace(null, "net.wg.gui.battle.views.vehicleMarkers:VehicleMarker"), "updateIconColor"), 0
+
+      returnvoid
+     end ; code
+     end ; body
+    end ; method
+   end ; trait
  end ; instance
  cinit
   refid "net.wg.gui.battle.views.vehicleMarkers:VehicleMarker/class/init"
