diff -Naur battle.orig-0.orig/net/wg/data/constants/Linkages.class.asasm battle.orig-0/net/wg/data/constants/Linkages.class.asasm
--- battle.orig-0.orig/net/wg/data/constants/Linkages.class.asasm	2022-08-28 23:51:04.477584800 +0300
+++ battle.orig-0/net/wg/data/constants/Linkages.class.asasm	2022-08-28 23:52:35.659861900 +0300
@@ -2380,7 +2380,7 @@
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_ITEM_LABEL") slotid 275 type QName(PackageNamespace(""), "String") value Utf8("battleProgressItemLabelUI") end
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_ITEM_LINE") slotid 276 type QName(PackageNamespace(""), "String") value Utf8("battleProgressItemLineUI") end
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_TASK_SEPARATOR") slotid 277 type QName(PackageNamespace(""), "String") value Utf8("battleProgressTaskSeparatorUI") end
- trait const QName(PackageNamespace(""), "CAPTURE_BAR_LINKAGE") slotid 278 type QName(PackageNamespace(""), "String") value Utf8("TeamCaptureBarUI") end
+ trait slot QName(PackageNamespace(""), "CAPTURE_BAR_LINKAGE") slotid 278 type QName(PackageNamespace(""), "String") value Utf8("TeamCaptureBarUI") end
  trait const QName(PackageNamespace(""), "VEH_CMP_WINDOW_BG_UI") slotid 279 type QName(PackageNamespace(""), "String") value Utf8("WindowBgUI") end
  trait const QName(PackageNamespace(""), "VEH_CMP_BUBBLE_UI") slotid 280 type QName(PackageNamespace(""), "String") value Utf8("VehCompareBubbleUI") end
  trait const QName(PackageNamespace(""), "DIP_ENGINE_DAMAGED") slotid 281 type QName(PackageNamespace(""), "String") value Utf8("DipEngineDamagedUI") end

