diff -Naur battle.orig-0.orig/net/wg/data/constants/Linkages.class.asasm battle.orig-0/net/wg/data/constants/Linkages.class.asasm
--- battle.orig-0.orig/net/wg/data/constants/Linkages.class.asasm	2022-09-21 22:29:46.575311200 +0300
+++ battle.orig-0/net/wg/data/constants/Linkages.class.asasm	2022-09-21 22:30:44.655056400 +0300
@@ -2405,7 +2405,7 @@
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_ITEM_LABEL") slotid 276 type QName(PackageNamespace(""), "String") value Utf8("battleProgressItemLabelUI") end
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_ITEM_LINE") slotid 277 type QName(PackageNamespace(""), "String") value Utf8("battleProgressItemLineUI") end
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_TASK_SEPARATOR") slotid 278 type QName(PackageNamespace(""), "String") value Utf8("battleProgressTaskSeparatorUI") end
- trait const QName(PackageNamespace(""), "CAPTURE_BAR_LINKAGE") slotid 279 type QName(PackageNamespace(""), "String") value Utf8("TeamCaptureBarUI") end
+ trait slot QName(PackageNamespace(""), "CAPTURE_BAR_LINKAGE") slotid 279 type QName(PackageNamespace(""), "String") value Utf8("TeamCaptureBarUI") end
  trait const QName(PackageNamespace(""), "EVENT_CAPTURE_BAR_LINKAGE") slotid 280 type QName(PackageNamespace(""), "String") value Utf8("EventTeamCaptureBarUI") end
  trait const QName(PackageNamespace(""), "VEH_CMP_WINDOW_BG_UI") slotid 281 type QName(PackageNamespace(""), "String") value Utf8("WindowBgUI") end
  trait const QName(PackageNamespace(""), "VEH_CMP_BUBBLE_UI") slotid 282 type QName(PackageNamespace(""), "String") value Utf8("VehCompareBubbleUI") end
