diff -Naur lobby.orig-0.orig/net/wg/gui/lobby/hangar/Hangar.class.asasm lobby.orig-0/net/wg/gui/lobby/hangar/Hangar.class.asasm
--- lobby.orig-0.orig/net/wg/gui/lobby/hangar/Hangar.class.asasm	2019-09-10 14:50:20.483664900 +0300
+++ lobby.orig-0/net/wg/gui/lobby/hangar/Hangar.class.asasm	2019-09-10 15:24:35.942311100 +0300
@@ -108,6 +108,24 @@
   trait slot QName(PackageNamespace(""), "switchModePanel") type QName(PackageNamespace("net.wg.gui.lobby.hangar"), "SwitchModePanel") end
   trait slot QName(PackageNamespace(""), "crewBG") type QName(PackageNamespace("flash.display"), "Sprite") end
   trait slot QName(PackageNamespace(""), "teaser") type QName(PackageNamespace("net.wg.gui.lobby.post"), "Teaser") end
+  trait getter QName(PackageNamespace(""), "xfw_header")
+   method
+    returns QName(PackageNamespace("net.wg.gui.lobby.hangar"), "HangarHeader")
+    body
+     maxstack 1
+     localcount 1
+     initscopedepth 11
+     maxscopedepth 12
+     code
+      getlocal0
+      pushscope
+      getlocal0
+      getproperty         QName(PrivateNamespace(null, "net.wg.gui.lobby.hangar:Hangar"), "_header")
+      returnvalue
+     end ; code
+    end ; body
+   end ; method
+  end ; trait
   trait slot QName(PackageNamespace(""), "lootboxesEntrancePoint") type QName(PackageNamespace("net.wg.gui.lobby.hangar"), "LootboxesEntrancePointWidget") end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.hangar:Hangar"), "_header") type QName(PackageNamespace("net.wg.gui.lobby.hangar"), "HangarHeader") end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.hangar:Hangar"), "_carousel") type QName(PackageNamespace("net.wg.gui.lobby.hangar.tcarousel"), "TankCarousel") end
