diff -Naur lobby.orig-0.orig/net/wg/gui/components/tooltips/ToolTipBase.class.asasm lobby.orig-0/net/wg/gui/components/tooltips/ToolTipBase.class.asasm
--- lobby.orig-0.orig/net/wg/gui/components/tooltips/ToolTipBase.class.asasm	2020-07-27 15:01:57.220559800 +0300
+++ lobby.orig-0/net/wg/gui/components/tooltips/ToolTipBase.class.asasm	2020-07-27 15:16:06.673348900 +0300
@@ -78,7 +78,7 @@
   trait slot QName(PrivateNamespace(null, "net.wg.gui.components.tooltips:ToolTipBase/instance"), "_timeStartBuild") type QName(PackageNamespace(""), "Number") value Integer(0) end
   trait const QName(PrivateNamespace(null, "net.wg.gui.components.tooltips:ToolTipBase/instance"), "CURSOR_OFFSET") type QName(PackageNamespace(""), "Number") value Integer(10) end
   trait const QName(PrivateNamespace(null, "net.wg.gui.components.tooltips:ToolTipBase/instance"), "BORDER_OFFSET") type QName(PackageNamespace(""), "Number") value Integer(15) end
-  trait const QName(PrivateNamespace(null, "net.wg.gui.components.tooltips:ToolTipBase/instance"), "SHOW_DELAY") type QName(PackageNamespace(""), "Number") value Integer(400) end
+  trait const QName(PrivateNamespace(null, "net.wg.gui.components.tooltips:ToolTipBase/instance"), "SHOW_DELAY") type QName(PackageNamespace(""), "Number") value Integer(300) end
   trait const QName(PrivateNamespace(null, "net.wg.gui.components.tooltips:ToolTipBase/instance"), "LIMIT_BUILD_TIME") type QName(PackageNamespace(""), "Number") value Double(500) end
   trait method QName(PackageNamespace(""), "toString") flag OVERRIDE
    method
diff -Naur lobby.orig-0.orig/net/wg/infrastructure/base/meta/impl/ToolTipMgrMeta.class.asasm lobby.orig-0/net/wg/infrastructure/base/meta/impl/ToolTipMgrMeta.class.asasm
--- lobby.orig-0.orig/net/wg/infrastructure/base/meta/impl/ToolTipMgrMeta.class.asasm	2020-07-27 15:01:56.650240300 +0300
+++ lobby.orig-0/net/wg/infrastructure/base/meta/impl/ToolTipMgrMeta.class.asasm	2020-07-27 15:16:06.688976300 +0300
@@ -23,6 +23,7 @@
     end ; code
    end ; body
   end ; method
+  trait slot QName(PackageNamespace(""), "xvm_hide") type QName(PackageNamespace(""), "Function") value Null() end
   trait slot QName(PackageNamespace(""), "onCreateComplexTooltip") type QName(PackageNamespace(""), "Function") end
   trait slot QName(PackageNamespace(""), "onCreateTypedTooltip") type QName(PackageNamespace(""), "Function") end
   trait slot QName(PackageNamespace(""), "onHideTooltip") type QName(PackageNamespace(""), "Function") end
diff -Naur lobby.orig-0.orig/net/wg/infrastructure/managers/impl/ToolTipManagerLobby.class.asasm lobby.orig-0/net/wg/infrastructure/managers/impl/ToolTipManagerLobby.class.asasm
--- lobby.orig-0.orig/net/wg/infrastructure/managers/impl/ToolTipManagerLobby.class.asasm	2020-07-27 15:01:56.665863400 +0300
+++ lobby.orig-0/net/wg/infrastructure/managers/impl/ToolTipManagerLobby.class.asasm	2020-07-27 15:17:09.409063500 +0300
@@ -362,15 +362,15 @@
     initproperty        QName(PrivateNamespace(null, "net.wg.infrastructure.managers.impl:ToolTipManagerLobby"), "NUM_SEPARATOR_POOL_ITEMS")
 
     findproperty        QName(PrivateNamespace(null, "net.wg.infrastructure.managers.impl:ToolTipManagerLobby"), "NUM_IMAGE_TEXT_BLOCK_POOL_ITEMS")
-    pushbyte            15
+    pushbyte            30
     initproperty        QName(PrivateNamespace(null, "net.wg.infrastructure.managers.impl:ToolTipManagerLobby"), "NUM_IMAGE_TEXT_BLOCK_POOL_ITEMS")
 
     findproperty        QName(PrivateNamespace(null, "net.wg.infrastructure.managers.impl:ToolTipManagerLobby"), "NUM_TEXT_PARAMETER_BLOCK_POOL_ITEMS")
-    pushbyte            20
+    pushbyte            30
     initproperty        QName(PrivateNamespace(null, "net.wg.infrastructure.managers.impl:ToolTipManagerLobby"), "NUM_TEXT_PARAMETER_BLOCK_POOL_ITEMS")
 
     findproperty        QName(PrivateNamespace(null, "net.wg.infrastructure.managers.impl:ToolTipManagerLobby"), "NUM_TEXT_PARAMETER_BLOCK_WITH_ICON_POOL_ITEMS")
-    pushbyte            15
+    pushbyte            30
     initproperty        QName(PrivateNamespace(null, "net.wg.infrastructure.managers.impl:ToolTipManagerLobby"), "NUM_TEXT_PARAMETER_BLOCK_WITH_ICON_POOL_ITEMS")
 
     findproperty        QName(PrivateNamespace(null, "net.wg.infrastructure.managers.impl:ToolTipManagerLobby"), "NUM_STATUS_DELTA_BLOCK_POOL_ITEMS")
diff -Naur lobby.orig-0.orig/net/wg/infrastructure/managers/impl/TooltipMgr/ToolTipManagerBase.class.asasm lobby.orig-0/net/wg/infrastructure/managers/impl/TooltipMgr/ToolTipManagerBase.class.asasm
--- lobby.orig-0.orig/net/wg/infrastructure/managers/impl/TooltipMgr/ToolTipManagerBase.class.asasm	2020-07-27 15:01:56.665863400 +0300
+++ lobby.orig-0/net/wg/infrastructure/managers/impl/TooltipMgr/ToolTipManagerBase.class.asasm	2020-07-27 15:16:06.711106700 +0300
@@ -436,6 +436,15 @@
       pushfalse
       initproperty        QName(PrivateNamespace(null, "net.wg.infrastructure.managers.impl.TooltipMgr:ToolTipManagerBase"), "_hideInProgress")
 
+      getlocal0
+      getproperty         QName(PackageNamespace(""), "xvm_hide")
+      pushnull
+      ifeq                XFW1
+
+      getlocal0
+      callpropvoid        QName(PackageNamespace(""), "xvm_hide"), 0
+
+XFW1:
       returnvoid
      end ; code
     end ; body
