--- lobby.orig-0.orig/net/wg/data/constants/Linkages.class.asasm	2020-06-06 23:05:56.119699000 +0300
+++ lobby.orig-0/net/wg/data/constants/Linkages.class.asasm	2020-06-06 23:07:19.805401100 +0300
@@ -1987,7 +1987,7 @@
  trait const QName(PackageNamespace(""), "CHART_STATISTICS_VIEW") slotid 182 type QName(PackageNamespace(""), "String") value Utf8("ChartsStatisticsView_UI") end
  trait const QName(PackageNamespace(""), "STATISTICS_DASH_LINE_TEXT_ITEM_RENDERER") slotid 183 type QName(PackageNamespace(""), "String") value Utf8("StatisticsDashLineTextItemIRenderer_UI") end
  trait const QName(PackageNamespace(""), "TECHNIQUE_STATISTICS_PERCENT_SIGN") slotid 184 type QName(PackageNamespace(""), "String") value Utf8("TechniqueStatisticsPercentSign_UI") end
- trait const QName(PackageNamespace(""), "TECHNIQUE_STATISTIC_TAB") slotid 185 type QName(PackageNamespace(""), "String") value Utf8("TechniqueStatisticTab_UI") end
+ trait slot QName(PackageNamespace(""), "TECHNIQUE_STATISTIC_TAB") slotid 185 type QName(PackageNamespace(""), "String") value Utf8("TechniqueStatisticTab_UI") end
  trait const QName(PackageNamespace(""), "TECHNIQUE_ACHIEVEMENT_TAB") slotid 186 type QName(PackageNamespace(""), "String") value Utf8("TechniqueAchievementTab_UI") end
  trait const QName(PackageNamespace(""), "DETAILED_STATISTICS_ROOT_UNIT") slotid 187 type QName(PackageNamespace(""), "String") value Utf8("DetailedStatisticsRootUnitUI") end
  trait const QName(PackageNamespace(""), "DEFAULT_HELP_BTN_CONTROLLER_LNK") slotid 188 type QName(PackageNamespace(""), "String") value Utf8("net.wg.infrastructure.tutorial.helpBtnControllers.TutorialWindowHelpBtnCtrllr") end
