diff -Naur lobby.orig-0.orig/net/wg/infrastructure/managers/impl/ImageManager/ImageData.class.asasm lobby.orig-0/net/wg/infrastructure/managers/impl/ImageManager/ImageData.class.asasm
--- lobby.orig-0.orig/net/wg/infrastructure/managers/impl/ImageManager/ImageData.class.asasm	2017-07-18 22:45:15.821059600 +0300
+++ lobby.orig-0/net/wg/infrastructure/managers/impl/ImageManager/ImageData.class.asasm	2017-07-18 22:50:20.985109200 +0300
@@ -465,7 +465,7 @@
       getlex              QName(PackageNamespace(""), "DebugUtils")
       getlocal1
       callproperty        QName(PackageNamespace(""), "toString"), 0
-      callpropvoid        QName(PackageNamespace(""), "LOG_ERROR"), 1
+      callpropvoid        QName(PackageNamespace(""), "LOG_WARNING"), 1
 
       findpropstrict      QName(PackageNamespace(""), "dispatchEvent")
       getlocal1
