diff -Naur battleVehicleMarkersApp.orig-0.orig/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.class.asasm battleVehicleMarkersApp.orig-0/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.class.asasm
--- battleVehicleMarkersApp.orig-0.orig/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.class.asasm	2023-03-11 22:00:53.797565500 +0300
+++ battleVehicleMarkersApp.orig-0/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.class.asasm	2023-03-11 22:01:36.084409900 +0300
@@ -2992,6 +2992,13 @@
       pushscope
 
       getlocal0
+      callproperty        QName(PackageNamespace(""), "xvm_active"), 0
+      iffalse             L6
+
+      returnvoid
+
+L6:
+      getlocal0
       getlocal0
       callproperty        QName(ProtectedNamespace("net.wg.gui.battle.views.vehicleMarkers:VehicleMarker"), "updatePartsVisibility"), 0
       callpropvoid        QName(PrivateNamespace(null, "net.wg.gui.battle.views.vehicleMarkers:VehicleMarker"), "layoutParts"), 1
@@ -4428,6 +4435,26 @@
      end ; code
     end ; body
    end ; method
+  end ; trait
+  trait method QName(PackageNamespace(""), "xvm_active")
+   method
+    name "xvm_active"
+    refid "net.wg.gui.battle.views.vehicleMarkers:VehicleMarker/instance/xvm_active"
+    returns QName(PackageNamespace(""), "Boolean")
+    body
+     maxstack 1
+     localcount 1
+     initscopedepth 0
+     maxscopedepth 1
+     code
+      getlocal0
+      pushscope
+
+      pushfalse
+      returnvalue
+     end ; code
+    end ; body
+   end ; method
   end ; trait
  end ; instance
  cinit
