diff -Naur battle.orig-0.orig/net/wg/gui/battle/ranked/stats/components/playersPanel/list/RankedPlayersPanelListLeft.class.asasm battle.orig-0/net/wg/gui/battle/ranked/stats/components/playersPanel/list/RankedPlayersPanelListLeft.class.asasm
--- battle.orig-0.orig/net/wg/gui/battle/ranked/stats/components/playersPanel/list/RankedPlayersPanelListLeft.class.asasm       2025-07-23 01:34:03.458814900 +0300
+++ battle.orig-0/net/wg/gui/battle/ranked/stats/components/playersPanel/list/RankedPlayersPanelListLeft.class.asasm   2025-07-23 03:05:12.596351100 +0300
@@ -38,7 +38,7 @@
       getlocal0
       pushscope

-      getlex              QName(PrivateNamespace(null, "net.wg.gui.battle.ranked.stats.components.playersPanel.list:RankedPlayersPanelListLeft"), "LINKAGE")
+      getlex              QName(PackageNamespace(""), "LINKAGE")
       returnvalue
      end ; code
     end ; body
@@ -222,7 +222,7 @@
     getlocal0
     pushscope

-    findproperty        QName(PrivateNamespace(null, "net.wg.gui.battle.ranked.stats.components.playersPanel.list:RankedPlayersPanelListLeft"), "LINKAGE")
+    findproperty        QName(PackageNamespace(""), "LINKAGE")
     pushstring          "RankedPlayersPanelListItemLeftUI"
-    initproperty        QName(PrivateNamespace(null, "net.wg.gui.battle.ranked.stats.components.playersPanel.list:RankedPlayersPanelListLeft"), "LINKAGE")
+    initproperty        QName(PackageNamespace(""), "LINKAGE")

@@ -234,6 +234,6 @@
    end ; code
   end ; body
  end ; method
- trait const QName(PrivateNamespace(null, "net.wg.gui.battle.ranked.stats.components.playersPanel.list:RankedPlayersPanelListLeft"), "LINKAGE") slotid 1 type QName(PackageNamespace(""), "String") value Utf8("RankedPlayersPanelListItemLeftUI") end
+ trait const QName(PackageNamespace(""), "LINKAGE") slotid 1 type QName(PackageNamespace(""), "String") value Utf8("RankedPlayersPanelListItemLeftUI") end
  trait const QName(PrivateNamespace(null, "net.wg.gui.battle.ranked.stats.components.playersPanel.list:RankedPlayersPanelListLeft"), "WIDTH") slotid 2 type QName(PackageNamespace(""), "uint") value Integer(485) end
 end ; class
diff -Naur battle.orig-0.orig/net/wg/gui/battle/ranked/stats/components/playersPanel/list/RankedPlayersPanelListRight.class.asasm battle.orig-0/net/wg/gui/battle/ranked/stats/components/playersPanel/list/RankedPlayersPanelListRight.class.asasm
--- battle.orig-0.orig/net/wg/gui/battle/ranked/stats/components/playersPanel/list/RankedPlayersPanelListRight.class.asasm      2025-07-23 01:34:07.947144000 +0300
+++ battle.orig-0/net/wg/gui/battle/ranked/stats/components/playersPanel/list/RankedPlayersPanelListRight.class.asasm  2025-07-23 03:05:45.551796300 +0300
@@ -36,7 +36,7 @@
       getlocal0
       pushscope

-      getlex              QName(PrivateNamespace(null, "net.wg.gui.battle.ranked.stats.components.playersPanel.list:RankedPlayersPanelListRight"), "LINKAGE")
+      getlex              QName(PackageNamespace(""), "LINKAGE")
       returnvalue
      end ; code
     end ; body
@@ -92,13 +92,13 @@
     getlocal0
     pushscope

-    findproperty        QName(PrivateNamespace(null, "net.wg.gui.battle.ranked.stats.components.playersPanel.list:RankedPlayersPanelListRight"), "LINKAGE")
+    findproperty        QName(PackageNamespace(""), "LINKAGE")
     pushstring          "RankedPlayersPanelListItemRightUI"
-    initproperty        QName(PrivateNamespace(null, "net.wg.gui.battle.ranked.stats.components.playersPanel.list:RankedPlayersPanelListRight"), "LINKAGE")
+    initproperty        QName(PackageNamespace(""), "LINKAGE")

     returnvoid
    end ; code
   end ; body
  end ; method
- trait const QName(PrivateNamespace(null, "net.wg.gui.battle.ranked.stats.components.playersPanel.list:RankedPlayersPanelListRight"), "LINKAGE") slotid 1 type QName(PackageNamespace(""), "String") value Utf8("RankedPlayersPanelListItemRightUI") end
+ trait const QName(PackageNamespace(""), "LINKAGE") slotid 1 type QName(PackageNamespace(""), "String") value Utf8("RankedPlayersPanelListItemRightUI") end
 end ; class
