diff -Naur battle.orig-0.orig/net/wg/data/constants/Linkages.class.asasm battle.orig-0/net/wg/data/constants/Linkages.class.asasm
--- battle.orig-0.orig/net/wg/data/constants/Linkages.class.asasm	2023-09-21 11:35:38.188126700 +0300
+++ battle.orig-0/net/wg/data/constants/Linkages.class.asasm	2023-09-21 11:37:06.955699800 +0300
@@ -2467,7 +2467,7 @@
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_ITEM_LABEL") slotid 282 type QName(PackageNamespace(""), "String") value Utf8("battleProgressItemLabelUI") end
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_ITEM_LINE") slotid 283 type QName(PackageNamespace(""), "String") value Utf8("battleProgressItemLineUI") end
  trait const QName(PackageNamespace(""), "BATTLE_PROGRESS_TASK_SEPARATOR") slotid 284 type QName(PackageNamespace(""), "String") value Utf8("battleProgressTaskSeparatorUI") end
- trait const QName(PackageNamespace(""), "CAPTURE_BAR_LINKAGE") slotid 285 type QName(PackageNamespace(""), "String") value Utf8("TeamCaptureBarUI") end
+ trait slot QName(PackageNamespace(""), "CAPTURE_BAR_LINKAGE") slotid 285 type QName(PackageNamespace(""), "String") value Utf8("TeamCaptureBarUI") end
  trait const QName(PackageNamespace(""), "EVENT_CAPTURE_BAR_LINKAGE") slotid 286 type QName(PackageNamespace(""), "String") value Utf8("EventTeamCaptureBarUI") end
  trait const QName(PackageNamespace(""), "VEH_CMP_WINDOW_BG_UI") slotid 287 type QName(PackageNamespace(""), "String") value Utf8("WindowBgUI") end
  trait const QName(PackageNamespace(""), "VEH_CMP_BUBBLE_UI") slotid 288 type QName(PackageNamespace(""), "String") value Utf8("VehCompareBubbleUI") end
