diff -Naur app.orig/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as app/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as
--- app.orig/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as	2014-09-23 14:18:22.977510700 +0300
+++ app/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as	2014-10-04 16:58:04.073314900 +0300
@@ -599,7 +599,7 @@
                                 _loc12_ = _loc7_[_loc17_];
                                 if(!(_loc12_ == null) && _loc12_.index == _loc16_)
                                 {
-                                    _loc4_ = param2?ADG_ItemLevelsBuilder.makeByNext(_loc11_,_loc12_):ADG_ItemLevelsBuilder.makeByTop(_loc11_,_loc12_);
+                                    _loc4_ = param2?LevelLine.makeByNext(_loc11_,_loc12_):LevelLine.makeByTop(_loc11_,_loc12_);
                                     _loc3_.push(_loc4_);
                                     break;
                                 }
