diff -Naur app.orig/App.as app/App.as
--- app.orig/App.as	2014-12-22 10:19:16.496630300 +0200
+++ app/App.as	2015-01-08 15:24:09.373222300 +0200
@@ -1,5 +1,6 @@
 package
 {
+    $AppLinks;
     import net.wg.app.IApplication;
     import net.wg.infrastructure.base.meta.IGlobalVarsMgrMeta;
     import net.wg.infrastructure.managers.ISoundManager;
diff -Naur app.orig/net/wg/gui/components/advanced/TankIcon.as app/net/wg/gui/components/advanced/TankIcon.as
--- app.orig/net/wg/gui/components/advanced/TankIcon.as	2014-12-22 10:24:03.368314600 +0200
+++ app/net/wg/gui/components/advanced/TankIcon.as	2015-01-08 15:22:35.123078700 +0200
@@ -96,7 +96,7 @@
         
         private var _isPremiumDirty:Boolean = true;
         
-        private var DIRTY_FLAGS:Array = [this._showMultyXpDirty,this._showXpDirty,this._showNameDirty,this._nationDirty,this._imageDirty,this._tankTypeDirty,this._levelDirty,this._multyXpValDirty,this._xpValDirty,this._tankNameDirty,this._isRentDirty,this._isEliteDirty,this._isPremiumDirty];
+        public var DIRTY_FLAGS:Array = [_showMultyXpDirty,_showXpDirty,_showNameDirty,_nationDirty,_imageDirty,_tankTypeDirty,_levelDirty,_multyXpValDirty,_xpValDirty,_tankNameDirty,_isRentDirty,_isEliteDirty,_isPremiumDirty];
         
         override protected function onDispose() : void
         {
diff -Naur app.orig/net/wg/gui/components/common/video/SimpleVideoPlayer.as app/net/wg/gui/components/common/video/SimpleVideoPlayer.as
--- app.orig/net/wg/gui/components/common/video/SimpleVideoPlayer.as	2014-12-22 10:22:00.119991000 +0200
+++ app/net/wg/gui/components/common/video/SimpleVideoPlayer.as	2015-01-08 15:22:35.123078700 +0200
@@ -311,10 +311,10 @@
         }
     }
     
-    public function get metaData() : Object
+    /*public function get metaData() : Object
     {
         return this._metaData;
-    }
+    }*/
     
     public function get currentTime() : Number
     {
diff -Naur app.orig/net/wg/gui/components/controls/Carousel.as app/net/wg/gui/components/controls/Carousel.as
--- app.orig/net/wg/gui/components/controls/Carousel.as	2014-12-22 10:22:02.649243900 +0200
+++ app/net/wg/gui/components/controls/Carousel.as	2015-01-08 15:33:01.590327100 +0200
@@ -92,8 +92,28 @@
         protected var selectedItemRenderer:IListItemRenderer = null;
         
         private var scopeWidth:Number = 0;
+
+        // XVM
+        public function get xvm_scopeWidth():Number
+        {
+            return scopeWidth;
+        }
+        public function set xvm_scopeWidth(value:Number):void
+        {
+            scopeWidth = value;
+        }
         
         private var courseFactor:Number = 0;
+
+        // XVM
+        public function get xvm_courseFactor():Number
+        {
+            return courseFactor;
+        }
+        public function set xvm_courseFactor(value:Number):void
+        {
+            courseFactor = value;
+        }
         
         private var slideAcceleratorAfterDrag:Number = 16;
         
@@ -617,7 +637,7 @@
     
 }
 
-private function getCurrentFirstRendererOnAnim() : Number
+protected function getCurrentFirstRendererOnAnim() : Number
 {
     var _loc1_:* = NaN;
     if(!container || !_renderers)
@@ -748,7 +768,7 @@
 this.courseFactor = 0;
 }
 
-private function arrowSlide() : void
+protected function arrowSlide() : void
 {
 var _loc1_:* = NaN;
 var _loc2_:* = NaN;
diff -Naur app.orig/net/wg/gui/components/controls/ScrollPane.as app/net/wg/gui/components/controls/ScrollPane.as
--- app.orig/net/wg/gui/components/controls/ScrollPane.as	2014-12-22 10:21:42.212200400 +0200
+++ app/net/wg/gui/components/controls/ScrollPane.as	2015-01-08 15:22:35.143078700 +0200
@@ -196,7 +196,7 @@
         override protected function onDispose() : void
         {
             var _loc1_:DisplayObject = null;
-            this._isScrollBarHaveToBeShown = null;
+            this._isScrollBarHaveToBeShown = false;
             super.onDispose();
             this.thumbOffset = null;
             if(this._scrollBar)
diff -Naur app.orig/net/wg/gui/components/tooltips/helpers/Utils.as app/net/wg/gui/components/tooltips/helpers/Utils.as
--- app.orig/net/wg/gui/components/tooltips/helpers/Utils.as	2014-12-22 10:23:22.187196900 +0200
+++ app/net/wg/gui/components/tooltips/helpers/Utils.as	2015-01-08 15:22:35.153078700 +0200
@@ -71,7 +71,7 @@
         
         public var STATUS_INFO:String = "info";
         
-        public var allowStatuses:Array = [this.STATUS_ADDITIONAL,this.STATUS_RENTED,this.STATUS_CRITICAL,this.STATUS_WARNING,this.STATUS_INFO];
+        public var allowStatuses:Array = [STATUS_ADDITIONAL,STATUS_RENTED,STATUS_CRITICAL,STATUS_WARNING,STATUS_INFO];
         
         public var MARGIN_AFTER_BLOCK:Number = 15;
         
diff -Naur app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as app/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as
--- app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as	2014-12-22 10:27:15.083484200 +0200
+++ app/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as	2015-01-08 15:35:56.402800200 +0200
@@ -83,9 +83,9 @@
         
         private var filterDataInvalid:Boolean = false;
         
-        private var nationFilter:DropDownImageText;
+        private var nationFilter:net.wg.gui.components.controls.DropDownImageText;
         
-        private var tankFilter:DropDownImageText;
+        private var tankFilter:net.wg.gui.components.controls.DropDownImageText;
         
         private var checkBoxToMain:CheckBox;
         
@@ -110,8 +110,28 @@
         private var _vehiclesVOManager:VehicleCarouselVOManager = null;
         
         private var _slotForBuySlot:IListItemRenderer = null;
+
+        // XVM
+        public function get xvm_slotForBuySlot():IListItemRenderer
+        {
+            return _slotForBuySlot;
+        }
+        public function set xvm_slotForBuySlot(value:IListItemRenderer):void
+        {
+            _slotForBuySlot = value;
+        }
         
         private var _slotForBuyVehicle:IListItemRenderer = null;
+
+        // XVM
+        public function get xvm_slotForBuyVehicle():IListItemRenderer
+        {
+            return _slotForBuyVehicle;
+        }
+        public function set xvm_slotForBuyVehicle(value:IListItemRenderer):void
+        {
+            _slotForBuyVehicle = value;
+        }
         
         private var _updateInProgress:Boolean = false;
         
@@ -759,7 +779,7 @@
             this.checkBoxToMain.addEventListener(Event.SELECT,this.onFilterCheckBoxChanged);
         }
         
-        private function showHideFilters() : void
+        protected function showHideFilters() : void
         {
             updateVisibleSlotsCount();
             var _loc1_:Boolean = _visibleSlots < this._createdRendersListByCompDescrLength || !(this._createdRendersListByCompDescrLength == this._currentShowByCompactDescription.length);
diff -Naur app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as app/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as
--- app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as	2014-12-22 10:27:36.959671600 +0200
+++ app/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as	2015-01-08 15:22:35.173078700 +0200
@@ -24,9 +24,9 @@
         
         public static var FILTER_READY:String = "ready";
         
-        public var nationFilter:DropDownImageText;
+        public var nationFilter:net.wg.gui.components.controls.DropDownImageText;
         
-        public var tankFilter:DropDownImageText;
+        public var tankFilter:net.wg.gui.components.controls.DropDownImageText;
         
         public var checkBoxToMain:CheckBox;
         
diff -Naur app.orig/net/wg/gui/lobby/header/LobbyHeader.as app/net/wg/gui/lobby/header/LobbyHeader.as
--- app.orig/net/wg/gui/lobby/header/LobbyHeader.as	2014-12-22 10:26:54.338409900 +0200
+++ app/net/wg/gui/lobby/header/LobbyHeader.as	2015-01-08 15:25:58.083449300 +0200
@@ -67,6 +67,12 @@
         
         private var _headerButtonsHelper:HeaderButtonsHelper;
         
+        // XVM
+        public function get xvm_headerButtonsHelper():HeaderButtonsHelper
+        {
+            return _headerButtonsHelper;
+        }
+
         private var _isShowHelpLayout:Boolean = false;
         
         private var _bubbleTooltip:ToolTipComplex;
diff -Naur app.orig/net/wg/gui/lobby/profile/ProfileTabNavigator.as app/net/wg/gui/lobby/profile/ProfileTabNavigator.as
--- app.orig/net/wg/gui/lobby/profile/ProfileTabNavigator.as	2014-12-22 10:24:42.546232000 +0200
+++ app/net/wg/gui/lobby/profile/ProfileTabNavigator.as	2015-01-08 15:22:35.173078700 +0200
@@ -37,6 +37,12 @@
         public var viewStack:ResizableViewStack;
         
         private var initData:ProfileMenuInfoVO;
+
+        // XVM
+        public function get xvm_initData():ProfileMenuInfoVO
+        {
+            return initData;
+        }
         
         private var _sectionsDataUtil:LinkageUtils;
         
diff -Naur app.orig/net/wg/gui/messenger/ChannelComponent.as app/net/wg/gui/messenger/ChannelComponent.as
--- app.orig/net/wg/gui/messenger/ChannelComponent.as	2014-12-22 10:27:18.287804600 +0200
+++ app/net/wg/gui/messenger/ChannelComponent.as	2015-01-08 15:22:35.193078800 +0200
@@ -28,7 +28,7 @@
         
         public var messageAreaScrollBar:ScrollBar = null;
         
-        public var messageArea:TextAreaSimple = null;
+        public var messageArea:net.wg.gui.components.advanced.TextAreaSimple = null;
         
         public var messageInput:TextInput = null;
         
diff -Naur app.orig/net/wg/gui/notification/NotificationPopUpViewer.as app/net/wg/gui/notification/NotificationPopUpViewer.as
--- app.orig/net/wg/gui/notification/NotificationPopUpViewer.as	2014-12-22 10:28:40.234998500 +0200
+++ app/net/wg/gui/notification/NotificationPopUpViewer.as	2015-01-08 15:29:43.333898000 +0200
@@ -46,6 +46,12 @@
     private var maxAvailaleMessagesCount:uint = 5;
     
     private var popupClass:Class;
+
+    // XVM
+    public function set xvm_popupClass(value:Class):void
+    {
+        popupClass = value;
+    }
     
     private var animationManager:ExcludeTweenManager;
     
diff -Naur app.orig/net/wg/gui/prebattle/squad/SquadItemRenderer.as app/net/wg/gui/prebattle/squad/SquadItemRenderer.as
--- app.orig/net/wg/gui/prebattle/squad/SquadItemRenderer.as	2014-12-22 10:20:58.613841000 +0200
+++ app/net/wg/gui/prebattle/squad/SquadItemRenderer.as	2015-01-08 15:22:35.203078800 +0200
@@ -171,7 +171,7 @@
                 }
                 this.setSpeakers(this.model.isPlayerSpeaking,true);
                 textField.alpha = this.vehicleNameField.alpha = 1;
-                _loc1_ = this.model.getStateString();
+                var _loc1_:* = this.model.getStateString();
                 if(_loc1_ != PrebattleStateString.UNKNOWN)
                 {
                     this.statusString = _loc1_;
@@ -186,8 +186,8 @@
                         this.showToolTips();
                     }
                 }
-                _loc2_ = 4.290295975E9;
-                _loc3_ = _loc2_;
+                var _loc2_:* = 4.290295975E9;
+                var _loc3_:* = _loc2_;
                 this.updateVoiceWave();
                 _loc3_ = this.model.getCurrentColor();
                 if(!isNaN(_loc3_))
diff -Naur app.orig/net/wg/gui/rally/events/RallyViewsEvent.as app/net/wg/gui/rally/events/RallyViewsEvent.as
--- app.orig/net/wg/gui/rally/events/RallyViewsEvent.as	2014-12-22 10:26:19.526929100 +0200
+++ app/net/wg/gui/rally/events/RallyViewsEvent.as	2015-01-08 15:22:35.213078800 +0200
@@ -41,7 +41,7 @@
         
         public static var VEH_BTN_ROLL_OUT:String = "vehBtnRollOut";
         
-        public var data;
+        public var data:*;
         
         override public function clone() : Event
         {
diff -Naur app.orig/net/wg/infrastructure/managers/IContainerManager.as app/net/wg/infrastructure/managers/IContainerManager.as
--- app.orig/net/wg/infrastructure/managers/IContainerManager.as	2014-12-22 10:19:16.405621200 +0200
+++ app/net/wg/infrastructure/managers/IContainerManager.as	2015-01-08 15:22:35.213078800 +0200
@@ -15,6 +15,8 @@
         
         function isModalViewsExisting() : Boolean;
         
+        function get loader() : ILoaderManager;
+
         function set loader(param1:ILoaderManager) : void;
         
         function get lastFocusedView() : IView;
diff -Naur app.orig/net/wg/infrastructure/managers/impl/ContextMenuManager.as app/net/wg/infrastructure/managers/impl/ContextMenuManager.as
--- app.orig/net/wg/infrastructure/managers/impl/ContextMenuManager.as	2014-12-22 10:19:42.587239100 +0200
+++ app/net/wg/infrastructure/managers/impl/ContextMenuManager.as	2015-01-08 15:22:35.223078800 +0200
@@ -48,7 +48,7 @@
         
         override protected function setOptions(param1:ContextMenuOptionsVO) : void
         {
-            assert(this._currentMenu,"Current context menu is null in setOptions call");
+            //assert(this._currentMenu,"Current context menu is null in setOptions call");
             this._currentMenu.build(param1.options,this._currentMenu.clickPoint);
         }
         
diff -Naur app.orig/org/idmedia/as3commons/util/EntrySetIterator.as app/org/idmedia/as3commons/util/EntrySetIterator.as
--- app.orig/org/idmedia/as3commons/util/EntrySetIterator.as	2014-12-22 10:31:53.857358800 +0200
+++ app/org/idmedia/as3commons/util/EntrySetIterator.as	2015-01-08 15:22:35.223078800 +0200
@@ -45,7 +45,7 @@
             }
             else
             {
-                _loc1_ = this.current.getKey();
+                var _loc1_:* = this.current.getKey();
                 this.current = null;
                 this.s.removeEntryForKey(_loc1_);
                 return;
diff -Naur app.orig/org/idmedia/as3commons/util/HashMap.as app/org/idmedia/as3commons/util/HashMap.as
--- app.orig/org/idmedia/as3commons/util/HashMap.as	2014-12-22 10:28:55.513526200 +0200
+++ app/org/idmedia/as3commons/util/HashMap.as	2015-01-08 15:22:35.233078800 +0200
@@ -58,9 +58,9 @@
         this.value = param2;
     }
     
-    private var key;
+    private var key:*;
     
-    private var value;
+    private var value:*;
     
     public function getKey() : *
     {
diff -Naur app.orig/scaleform/clik/controls/ButtonGroup.as app/scaleform/clik/controls/ButtonGroup.as
--- app.orig/scaleform/clik/controls/ButtonGroup.as	2014-12-22 10:27:51.094084900 +0200
+++ app/scaleform/clik/controls/ButtonGroup.as	2015-01-08 15:22:35.243078800 +0200
@@ -68,6 +68,7 @@
                 _loc1_ = _loc2_ as DisplayObjectContainer;
                 return _loc1_;
             }
+            return null;
         }
         
         public function addButton(param1:Button) : void
diff -Naur app.orig/scaleform/clik/ui/InputDetails.as app/scaleform/clik/ui/InputDetails.as
--- app.orig/scaleform/clik/ui/InputDetails.as	2014-12-22 10:28:54.791454000 +0200
+++ app/scaleform/clik/ui/InputDetails.as	2015-01-08 15:22:35.243078800 +0200
@@ -20,7 +20,7 @@
         
         public var code:Number;
         
-        public var value;
+        public var value:*;
         
         public var navEquivalent:String;
         
