diff -Naur app.orig/App.as app/App.as
--- app.orig/App.as	2014-07-30 12:59:11.178935000 +0300
+++ app/App.as	2014-07-30 13:14:56.234856100 +0300
@@ -1,5 +1,6 @@
 package
 {
+    $AppLinks;
     import net.wg.app.IApplication;
     import net.wg.infrastructure.base.meta.IGlobalVarsMgrMeta;
     import net.wg.infrastructure.managers.ISoundManager;
diff -Naur app.orig/net/wg/gui/components/advanced/TankIcon.as app/net/wg/gui/components/advanced/TankIcon.as
--- app.orig/net/wg/gui/components/advanced/TankIcon.as	2014-07-30 13:01:00.705638300 +0300
+++ app/net/wg/gui/components/advanced/TankIcon.as	2014-07-30 13:14:56.244856700 +0300
@@ -90,7 +90,7 @@
         
         private var _isPremiumDirty:Boolean = true;
         
-        private var DIRTY_FLAGS:Array = [this._showMultyXpDirty,this._showXpDirty,this._showNameDirty,this._nationDirty,this._imageDirty,this._tankTypeDirty,this._levelDirty,this._multyXpValDirty,this._xpValDirty,this._tankNameDirty,this._isEliteDirty,this._isPremiumDirty];
+        public var DIRTY_FLAGS:Array = [_showMultyXpDirty,_showXpDirty,_showNameDirty,_nationDirty,_imageDirty,_tankTypeDirty,_levelDirty,_multyXpValDirty,_xpValDirty,_tankNameDirty,_isEliteDirty,_isPremiumDirty];
         
         override protected function onDispose() : void
         {
diff -Naur app.orig/net/wg/gui/components/common/video/SimpleVideoPlayer.as app/net/wg/gui/components/common/video/SimpleVideoPlayer.as
--- app.orig/net/wg/gui/components/common/video/SimpleVideoPlayer.as	2014-07-30 13:00:16.283507300 +0300
+++ app/net/wg/gui/components/common/video/SimpleVideoPlayer.as	2014-07-30 13:14:56.260857600 +0300
@@ -311,10 +311,10 @@
         }
     }
     
-    public function get metaData() : Object
-    {
-        return this._metaData;
-    }
+    //public function get metaData() : Object
+    //{
+    //    return this._metaData;
+    //}
     
     public function get currentTime() : Number
     {
diff -Naur app.orig/net/wg/gui/components/controls/Carousel.as app/net/wg/gui/components/controls/Carousel.as
--- app.orig/net/wg/gui/components/controls/Carousel.as	2014-07-30 13:00:17.635511000 +0300
+++ app/net/wg/gui/components/controls/Carousel.as	2014-08-24 00:39:38.336425700 +0300
@@ -88,7 +88,7 @@
         
         protected var selectedItemRenderer:IListItemRenderer = null;
         
-        private var scopeWidth:Number = 0;
+        protected var scopeWidth:Number = 0;
         
         private var courseFactor:Number = 0;
         
diff -Naur app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as app/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as
--- app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as	2014-07-30 13:02:03.247737100 +0300
+++ app/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as	2014-08-24 00:40:31.776500500 +0300
@@ -84,9 +84,9 @@
         
         private var filterDataInvalid:Boolean = false;
         
-        private var nationFilter:DropDownImageText;
+        private var nationFilter:net.wg.gui.components.controls.DropDownImageText;
         
-        private var tankFilter:DropDownImageText;
+        private var tankFilter:net.wg.gui.components.controls.DropDownImageText;
         
         private var checkBoxToMain:CheckBox;
         
diff -Naur app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as app/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as
--- app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as	2014-07-30 13:02:10.417747100 +0300
+++ app/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as	2014-07-30 13:14:56.280858700 +0300
@@ -22,9 +22,9 @@
         
         public static var FILTER_READY:String = "ready";
         
-        public var nationFilter:DropDownImageText;
+        public var nationFilter:net.wg.gui.components.controls.DropDownImageText;
         
-        public var tankFilter:DropDownImageText;
+        public var tankFilter:net.wg.gui.components.controls.DropDownImageText;
         
         public var checkBoxToMain:CheckBox;
         
diff -Naur app.orig/net/wg/gui/lobby/profile/ProfileTabNavigator.as app/net/wg/gui/lobby/profile/ProfileTabNavigator.as
--- app.orig/net/wg/gui/lobby/profile/ProfileTabNavigator.as	2014-07-30 13:01:13.029659700 +0300
+++ app/net/wg/gui/lobby/profile/ProfileTabNavigator.as	2014-07-30 13:14:56.289859300 +0300
@@ -28,7 +28,7 @@
         
         public var viewStack:ResizableViewStack;
         
-        private var initData:ProfileMenuInfoVO;
+        public var initData:ProfileMenuInfoVO;
         
         private var _sectionsDataUtil:SectionsDataUtil;
         
diff -Naur app.orig/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as app/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as
--- app.orig/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as	2014-07-30 13:01:49.717718200 +0300
+++ app/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as	2014-07-30 13:14:56.332861700 +0300
@@ -599,7 +599,7 @@
                                 _loc12_ = _loc7_[_loc17_];
                                 if(!(_loc12_ == null) && _loc12_.index == _loc16_)
                                 {
-                                    _loc4_ = param2?ADG_ItemLevelsBuilder.makeByNext(_loc11_,_loc12_):ADG_ItemLevelsBuilder.makeByTop(_loc11_,_loc12_);
+                                    _loc4_ = param2?LevelLine.makeByNext(_loc11_,_loc12_):LevelLine.makeByTop(_loc11_,_loc12_);
                                     _loc3_.push(_loc4_);
                                     break;
                                 }
diff -Naur app.orig/net/wg/gui/notification/NotificationPopUpViewer.as app/net/wg/gui/notification/NotificationPopUpViewer.as
--- app.orig/net/wg/gui/notification/NotificationPopUpViewer.as	2014-07-30 13:02:31.031780100 +0300
+++ app/net/wg/gui/notification/NotificationPopUpViewer.as	2014-07-30 13:14:56.342862300 +0300
@@ -45,7 +45,7 @@
     
     private var maxAvailaleMessagesCount:uint = 5;
     
-    private var popupClass:Class;
+    public var popupClass:Class;
     
     private var animationManager:ExcludeTweenManager;
     
diff -Naur app.orig/net/wg/gui/prebattle/squad/SquadItemRenderer.as app/net/wg/gui/prebattle/squad/SquadItemRenderer.as
--- app.orig/net/wg/gui/prebattle/squad/SquadItemRenderer.as	2014-07-30 12:59:55.573451000 +0300
+++ app/net/wg/gui/prebattle/squad/SquadItemRenderer.as	2014-07-30 13:14:56.353862900 +0300
@@ -171,7 +171,7 @@
                 }
                 this.setSpeakers(this.model.isPlayerSpeaking,true);
                 textField.alpha = this.vehicleNameField.alpha = 1;
-                _loc1_ = this.model.getStateString();
+                var _loc1_:* = this.model.getStateString();
                 if(_loc1_ != PrebattleStateString.UNKNOWN)
                 {
                     this.statusString = _loc1_;
@@ -186,8 +186,8 @@
                         this.showToolTips();
                     }
                 }
-                _loc2_ = 4.290295975E9;
-                _loc3_ = _loc2_;
+                var _loc2_:* = 4.290295975E9;
+                var _loc3_:* = _loc2_;
                 this.updateVoiceWave();
                 _loc3_ = this.model.getCurrentColor();
                 if(!isNaN(_loc3_))
diff -Naur app.orig/net/wg/gui/prebattle/squad/SquadWindow.as app/net/wg/gui/prebattle/squad/SquadWindow.as
--- app.orig/net/wg/gui/prebattle/squad/SquadWindow.as	2014-07-30 13:01:21.769671900 +0300
+++ app/net/wg/gui/prebattle/squad/SquadWindow.as	2014-07-30 13:14:56.363863500 +0300
@@ -68,7 +68,7 @@
             if(param3)
             {
                 _loc4_ = new PlayerPrbInfoVO(param3);
-                _loc5_ = 0;
+                var _loc5_:* = 0;
                 while(_loc5_ < this.memberList.dataProvider.length)
                 {
                     _loc6_ = this.memberList.dataProvider.requestItemAt(_loc5_) as PlayerPrbInfoVO;
diff -Naur app.orig/net/wg/infrastructure/managers/IContainerManager.as app/net/wg/infrastructure/managers/IContainerManager.as
--- app.orig/net/wg/infrastructure/managers/IContainerManager.as	2014-07-30 12:59:11.084932900 +0300
+++ app/net/wg/infrastructure/managers/IContainerManager.as	2014-07-30 13:14:56.369863800 +0300
@@ -15,6 +15,8 @@
         
         function isModalViewsExisting() : Boolean;
         
+        function get loader() : ILoaderManager;
+
         function set loader(param1:ILoaderManager) : void;
         
         function get lastFocusedView() : IView;
diff -Naur app.orig/scaleform/clik/controls/ButtonGroup.as app/scaleform/clik/controls/ButtonGroup.as
--- app.orig/scaleform/clik/controls/ButtonGroup.as	2014-07-30 13:02:14.747753200 +0300
+++ app/scaleform/clik/controls/ButtonGroup.as	2014-07-30 13:14:56.376864200 +0300
@@ -68,6 +68,7 @@
                 _loc1_ = _loc2_ as DisplayObjectContainer;
                 return _loc1_;
             }
+            return null;
         }
         
         public function addButton(param1:Button) : void
diff -Naur app.orig/scaleform/clik/ui/InputDetails.as app/scaleform/clik/ui/InputDetails.as
--- app.orig/scaleform/clik/ui/InputDetails.as	2014-07-30 13:02:31.241780400 +0300
+++ app/scaleform/clik/ui/InputDetails.as	2014-07-30 13:14:56.382864600 +0300
@@ -20,7 +20,7 @@
         
         public var code:Number;
         
-        public var value;
+        public var value:*;
         
         public var navEquivalent:String;
         
