diff -Naur app.orig/App.as app/App.as
--- app.orig/App.as	2014-07-12 13:59:31.472880400 +0300
+++ app/App.as	2014-07-11 00:25:26.476656100 +0300
@@ -1,6 +1,5 @@
 package 
 {
-   $AppLinks;
    import net.wg.app.IApplication;
    import net.wg.infrastructure.base.meta.IGlobalVarsMgrMeta;
    import net.wg.infrastructure.managers.ISoundManager;
diff -Naur app.orig/net/wg/data/constants/Errors.as app/net/wg/data/constants/Errors.as
--- app.orig/net/wg/data/constants/Errors.as	2014-07-11 00:25:26.186655700 +0300
+++ app/net/wg/data/constants/Errors.as	2014-07-12 14:01:47.943071400 +0300
@@ -29,6 +29,6 @@
       
       public static const MTHD_CORRUPT_INVOKE:String = "method invoking after object destruction";
       
-      public static const INVALID_FOCUS_USING:String;
+      public static const INVALID_FOCUS_USING:String = "";
    }
 }
diff -Naur app.orig/net/wg/data/daapi/base/DAAPIDataClass.as app/net/wg/data/daapi/base/DAAPIDataClass.as
--- app.orig/net/wg/data/daapi/base/DAAPIDataClass.as	2014-07-11 00:26:18.268729800 +0300
+++ app/net/wg/data/daapi/base/DAAPIDataClass.as	2014-07-12 14:02:19.923116200 +0300
@@ -156,7 +156,7 @@
          return true;
       }
       
-      function getHash() : Object {
+      public function getHash() : Object {
          return this._hash;
       }
       
diff -Naur app.orig/scaleform/clik/managers/FocusHandler.as app/scaleform/clik/managers/FocusHandler.as
--- app.orig/scaleform/clik/managers/FocusHandler.as	2014-07-11 00:26:11.368720200 +0300
+++ app/scaleform/clik/managers/FocusHandler.as	2014-07-12 14:01:09.443017500 +0300
@@ -1,5 +1,6 @@
 package scaleform.clik.managers
 {
+   import flash.errors.*;
    import flash.display.Stage;
    import scaleform.clik.core.UIComponent;
    import scaleform.gfx.FocusManager;
diff -Naur app.orig/scaleform/clik/ui/InputDetails.as app/scaleform/clik/ui/InputDetails.as
--- app.orig/scaleform/clik/ui/InputDetails.as	2014-07-11 00:39:55.823415200 +0300
+++ app/scaleform/clik/ui/InputDetails.as	2014-07-12 14:00:22.332951600 +0300
@@ -19,7 +19,7 @@
       
       public var code:Number;
       
-      public var value;
+      public var value:*;
       
       public var navEquivalent:String;
       
