diff -Naur app.orig/App.as app/App.as
--- app.orig/App.as	2014-09-06 22:42:06.126127000 +0300
+++ app/App.as	2014-09-06 22:49:46.886985000 +0300
@@ -1,5 +1,6 @@
 package
 {
+    $AppLinks;
     import net.wg.app.IApplication;
     import net.wg.infrastructure.base.meta.IGlobalVarsMgrMeta;
     import net.wg.infrastructure.managers.ISoundManager;
diff -Naur app.orig/net/wg/gui/components/advanced/TankIcon.as app/net/wg/gui/components/advanced/TankIcon.as
--- app.orig/net/wg/gui/components/advanced/TankIcon.as	2014-09-06 22:44:18.158930200 +0300
+++ app/net/wg/gui/components/advanced/TankIcon.as	2014-09-06 22:49:46.896985000 +0300
@@ -90,7 +90,7 @@
         
         private var _isPremiumDirty:Boolean = true;
         
-        private var DIRTY_FLAGS:Array = [this._showMultyXpDirty,this._showXpDirty,this._showNameDirty,this._nationDirty,this._imageDirty,this._tankTypeDirty,this._levelDirty,this._multyXpValDirty,this._xpValDirty,this._tankNameDirty,this._isEliteDirty,this._isPremiumDirty];
+        public var DIRTY_FLAGS:Array = [_showMultyXpDirty,_showXpDirty,_showNameDirty,_nationDirty,_imageDirty,_tankTypeDirty,_levelDirty,_multyXpValDirty,_xpValDirty,_tankNameDirty,_isEliteDirty,_isPremiumDirty];
         
         override protected function onDispose() : void
         {
diff -Naur app.orig/net/wg/gui/components/common/video/SimpleVideoPlayer.as app/net/wg/gui/components/common/video/SimpleVideoPlayer.as
--- app.orig/net/wg/gui/components/common/video/SimpleVideoPlayer.as	2014-09-06 22:43:22.754819500 +0300
+++ app/net/wg/gui/components/common/video/SimpleVideoPlayer.as	2014-09-06 22:49:46.906985000 +0300
@@ -311,10 +311,10 @@
         }
     }
     
-    public function get metaData() : Object
-    {
-        return this._metaData;
-    }
+    //public function get metaData() : Object
+    //{
+    //    return this._metaData;
+    //}
     
     public function get currentTime() : Number
     {
diff -Naur app.orig/net/wg/gui/components/controls/Carousel.as app/net/wg/gui/components/controls/Carousel.as
--- app.orig/net/wg/gui/components/controls/Carousel.as	2014-09-06 22:43:24.016822500 +0300
+++ app/net/wg/gui/components/controls/Carousel.as	2014-09-06 22:49:46.926985000 +0300
@@ -91,9 +91,9 @@
         
         protected var selectedItemRenderer:IListItemRenderer = null;
         
-        private var scopeWidth:Number = 0;
+        protected var scopeWidth:Number = 0;
         
-        private var courseFactor:Number = 0;
+        protected var courseFactor:Number = 0;
         
         private var slideAcceleratorAfterDrag:Number = 16;
         
@@ -608,7 +608,7 @@
     
 }
 
-private function getCurrentFirstRendererOnAnim() : Number
+protected function getCurrentFirstRendererOnAnim() : Number
 {
     var _loc1_:* = NaN;
     if(!container || !_renderers)
@@ -739,7 +739,7 @@
 this.courseFactor = 0;
 }
 
-private function arrowSlide() : void
+protected function arrowSlide() : void
 {
 var _loc1_:* = NaN;
 var _loc2_:* = NaN;
diff -Naur app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as app/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as
--- app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as	2014-09-06 22:45:42.815070800 +0300
+++ app/net/wg/gui/lobby/hangar/tcarousel/TankCarousel.as	2014-09-07 19:11:31.443703100 +0300
@@ -86,9 +86,9 @@
         
         private var filterDataInvalid:Boolean = false;
         
-        private var nationFilter:DropDownImageText;
+        private var nationFilter:net.wg.gui.components.controls.DropDownImageText;
         
-        private var tankFilter:DropDownImageText;
+        private var tankFilter:net.wg.gui.components.controls.DropDownImageText;
         
         private var checkBoxToMain:CheckBox;
         
@@ -112,9 +112,9 @@
         
         private var _vehiclesVOManager:VehicleCarouselVOManager = null;
         
-        private var _slotForBuySlot:IListItemRenderer = null;
+        protected var _slotForBuySlot:IListItemRenderer = null;
         
-        private var _slotForBuyVehicle:IListItemRenderer = null;
+        protected var _slotForBuyVehicle:IListItemRenderer = null;
         
         private var _updateInProgress:Boolean = false;
         
@@ -734,7 +734,7 @@
         this.checkBoxToMain.addEventListener(Event.SELECT,this.onFilterCheckBoxChanged);
     }
     
-    private function showHideFilters() : void
+    protected function showHideFilters() : void
     {
         updateVisibleSlotsCount();
         var _loc1_:Boolean = _visibleSlots < this._createdRendersListByCompDescrLength || !(this._createdRendersListByCompDescrLength == this._currentShowByCompactDescription.length);
diff -Naur app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as app/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as
--- app.orig/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as	2014-09-06 22:45:53.525102600 +0300
+++ app/net/wg/gui/lobby/hangar/tcarousel/TankCarouselFilters.as	2014-09-06 22:49:46.946985000 +0300
@@ -22,9 +22,9 @@
         
         public static var FILTER_READY:String = "ready";
         
-        public var nationFilter:DropDownImageText;
+        public var nationFilter:net.wg.gui.components.controls.DropDownImageText;
         
-        public var tankFilter:DropDownImageText;
+        public var tankFilter:net.wg.gui.components.controls.DropDownImageText;
         
         public var checkBoxToMain:CheckBox;
         
diff -Naur app.orig/net/wg/gui/lobby/profile/ProfileTabNavigator.as app/net/wg/gui/lobby/profile/ProfileTabNavigator.as
--- app.orig/net/wg/gui/lobby/profile/ProfileTabNavigator.as	2014-09-06 22:44:34.408953000 +0300
+++ app/net/wg/gui/lobby/profile/ProfileTabNavigator.as	2014-09-06 22:49:46.956985100 +0300
@@ -36,7 +36,7 @@
         
         public var viewStack:ResizableViewStack;
         
-        private var initData:ProfileMenuInfoVO;
+        public var initData:ProfileMenuInfoVO;
         
         private var _sectionsDataUtil:LinkageUtils;
         
diff -Naur app.orig/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as app/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as
--- app.orig/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as	2014-09-06 22:45:24.873042800 +0300
+++ app/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as	2014-09-06 22:49:46.966985100 +0300
@@ -599,7 +599,7 @@
                                 _loc12_ = _loc7_[_loc17_];
                                 if(!(_loc12_ == null) && _loc12_.index == _loc16_)
                                 {
-                                    _loc4_ = param2?ADG_ItemLevelsBuilder.makeByNext(_loc11_,_loc12_):ADG_ItemLevelsBuilder.makeByTop(_loc11_,_loc12_);
+                                    _loc4_ = param2?LevelLine.makeByNext(_loc11_,_loc12_):LevelLine.makeByTop(_loc11_,_loc12_);
                                     _loc3_.push(_loc4_);
                                     break;
                                 }
diff -Naur app.orig/net/wg/gui/notification/NotificationPopUpViewer.as app/net/wg/gui/notification/NotificationPopUpViewer.as
--- app.orig/net/wg/gui/notification/NotificationPopUpViewer.as	2014-09-06 22:46:21.175141300 +0300
+++ app/net/wg/gui/notification/NotificationPopUpViewer.as	2014-09-06 22:49:46.976985100 +0300
@@ -45,7 +45,7 @@
     
     private var maxAvailaleMessagesCount:uint = 5;
     
-    private var popupClass:Class;
+    public var popupClass:Class;
     
     private var animationManager:ExcludeTweenManager;
     
diff -Naur app.orig/net/wg/gui/prebattle/squad/SquadItemRenderer.as app/net/wg/gui/prebattle/squad/SquadItemRenderer.as
--- app.orig/net/wg/gui/prebattle/squad/SquadItemRenderer.as	2014-09-06 22:42:55.242674800 +0300
+++ app/net/wg/gui/prebattle/squad/SquadItemRenderer.as	2014-09-06 22:49:46.986985100 +0300
@@ -171,7 +171,7 @@
                 }
                 this.setSpeakers(this.model.isPlayerSpeaking,true);
                 textField.alpha = this.vehicleNameField.alpha = 1;
-                _loc1_ = this.model.getStateString();
+                var _loc1_:* = this.model.getStateString();
                 if(_loc1_ != PrebattleStateString.UNKNOWN)
                 {
                     this.statusString = _loc1_;
@@ -186,8 +186,8 @@
                         this.showToolTips();
                     }
                 }
-                _loc2_ = 4.290295975E9;
-                _loc3_ = _loc2_;
+                var _loc2_:* = 4.290295975E9;
+                var _loc3_:* = _loc2_;
                 this.updateVoiceWave();
                 _loc3_ = this.model.getCurrentColor();
                 if(!isNaN(_loc3_))
diff -Naur app.orig/net/wg/gui/rally/events/RallyViewsEvent.as app/net/wg/gui/rally/events/RallyViewsEvent.as
--- app.orig/net/wg/gui/rally/events/RallyViewsEvent.as	2014-09-06 22:45:18.543033900 +0300
+++ app/net/wg/gui/rally/events/RallyViewsEvent.as	2014-09-06 22:54:59.369757100 +0300
@@ -41,7 +41,7 @@
         
         public static var VEH_BTN_ROLL_OUT:String = "vehBtnRollOut";
         
-        public var data;
+        public var data:*;
         
         override public function clone() : Event
         {
diff -Naur app.orig/net/wg/infrastructure/managers/IContainerManager.as app/net/wg/infrastructure/managers/IContainerManager.as
--- app.orig/net/wg/infrastructure/managers/IContainerManager.as	2014-09-06 22:42:06.033121700 +0300
+++ app/net/wg/infrastructure/managers/IContainerManager.as	2014-09-06 22:49:46.996985100 +0300
@@ -15,6 +15,8 @@
         
         function isModalViewsExisting() : Boolean;
         
+        function get loader() : ILoaderManager;
+
         function set loader(param1:ILoaderManager) : void;
         
         function get lastFocusedView() : IView;
diff -Naur app.orig/scaleform/clik/controls/ButtonGroup.as app/scaleform/clik/controls/ButtonGroup.as
--- app.orig/scaleform/clik/controls/ButtonGroup.as	2014-09-06 22:45:59.735111300 +0300
+++ app/scaleform/clik/controls/ButtonGroup.as	2014-09-06 22:49:47.006985100 +0300
@@ -68,6 +68,7 @@
                 _loc1_ = _loc2_ as DisplayObjectContainer;
                 return _loc1_;
             }
+            return null;
         }
         
         public function addButton(param1:Button) : void
diff -Naur app.orig/scaleform/clik/ui/InputDetails.as app/scaleform/clik/ui/InputDetails.as
--- app.orig/scaleform/clik/ui/InputDetails.as	2014-09-06 22:46:26.835149200 +0300
+++ app/scaleform/clik/ui/InputDetails.as	2014-09-06 22:49:47.016985100 +0300
@@ -20,7 +20,7 @@
         
         public var code:Number;
         
-        public var value;
+        public var value:*;
         
         public var navEquivalent:String;
         
