diff -Naur battleVehicleMarkersApp.orig/App.as battleVehicleMarkersApp/App.as
--- battleVehicleMarkersApp.orig/App.as	2020-07-27 14:30:03.521556800 +0300
+++ battleVehicleMarkersApp/App.as	2020-07-27 14:41:27.935585500 +0300
@@ -1,5 +1,6 @@
 package
 {
+    $AppLinks;
     import net.wg.app.IApplication;
     import scaleform.gfx.Extensions;
     import net.wg.mock.MockApplication;
diff -Naur battleVehicleMarkersApp.orig/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.as battleVehicleMarkersApp/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.as
--- battleVehicleMarkersApp.orig/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.as	2020-07-27 14:30:03.956062400 +0300
+++ battleVehicleMarkersApp/net/wg/gui/battle/views/vehicleMarkers/VehicleMarker.as	2020-07-27 14:41:27.966835300 +0300
@@ -631,8 +631,16 @@
             this._stunSchemeName = VM_STUN_PREFIX + this._entityName + VM_STUN_POSTFIX;
         }
 
+        public function xvm_active() : Boolean
+        {
+            return false;
+        }
+
         private function updateMarkerSettings() : void
         {
+            if (xvm_active())
+              return;
+
             var _loc9_:Point = null;
             var _loc1_:Boolean = this.getIsPartVisible(ICON);
             var _loc2_:Boolean = this.getIsPartVisible(LEVEL);
@@ -880,6 +888,11 @@
             this._vmManager.drawWithCenterAlign(_loc1_,this.marker.vehicleTypeIcon.graphics,true,false,0,V_TYPE_ICON_Y);
         }
 
+        public function xfw_updateIconColor() : void
+        {
+            updateIconColor();
+        }
+
         private function updateIconColor() : void
         {
             this.vehicleIcon.transform.colorTransform = this._vmManager.getTransform(this._markerSchemeName);
diff -Naur battleVehicleMarkersApp.orig/net/wg/infrastructure/base/meta/impl/ClassManagerBattleMarkersMeta.as battleVehicleMarkersApp/net/wg/infrastructure/base/meta/impl/ClassManagerBattleMarkersMeta.as
--- battleVehicleMarkersApp.orig/net/wg/infrastructure/base/meta/impl/ClassManagerBattleMarkersMeta.as	2020-07-27 14:30:03.405670300 +0300
+++ battleVehicleMarkersApp/net/wg/infrastructure/base/meta/impl/ClassManagerBattleMarkersMeta.as	2020-07-27 14:41:28.020225800 +0300
@@ -146,7 +146,7 @@
 
         public static const NET_WG_GUI_BATTLE_VIEWS_VEHICLEMARKERS_VEHICLEICONANIMATION:Class = VehicleIconAnimation;
 
-        public static const NET_WG_GUI_BATTLE_VIEWS_VEHICLEMARKERS_VEHICLEMARKER:Class = VehicleMarker;
+        public static const NET_WG_GUI_BATTLE_VIEWS_VEHICLEMARKERS_VEHICLEMARKER:Class = net.wg.gui.battle.views.vehicleMarkers.VehicleMarker;
 
         public static const NET_WG_GUI_BATTLE_VIEWS_VEHICLEMARKERS_VEHICLEMARKERSCONSTANTS:Class = VehicleMarkersConstants;
 
diff -Naur battleVehicleMarkersApp.orig/scaleform/clik/ui/InputDetails.as battleVehicleMarkersApp/scaleform/clik/ui/InputDetails.as
--- battleVehicleMarkersApp.orig/scaleform/clik/ui/InputDetails.as	2020-07-27 14:30:04.430849800 +0300
+++ battleVehicleMarkersApp/scaleform/clik/ui/InputDetails.as	2020-07-27 14:41:28.035857000 +0300
@@ -7,7 +7,7 @@
 
         public var code:Number;
 
-        public var value;
+        public var value:*;
 
         public var navEquivalent:String;
 
