diff -Naur lobby.orig/App.as lobby/App.as
--- lobby.orig/App.as	2020-02-28 20:15:48.755028700 +0200
+++ lobby/App.as	2020-02-28 20:19:21.712029800 +0200
@@ -1,5 +1,6 @@
 package
 {
+    $AppLinks;
     import net.wg.app.IApplication;
     import scaleform.gfx.Extensions;
     import net.wg.mock.MockApplication;
@@ -43,7 +44,7 @@
         {
             if(ms_instance == null && (!Extensions.isScaleform || Extensions.isGFxPlayer))
             {
-                ms_instance = new MockApplication();
+                //ms_instance = new MockApplication();
             }
             return ms_instance;
         }
diff -Naur lobby.orig/net/wg/data/constants/Linkages.as lobby/net/wg/data/constants/Linkages.as
--- lobby.orig/net/wg/data/constants/Linkages.as	2020-02-28 20:15:55.426892200 +0200
+++ lobby/net/wg/data/constants/Linkages.as	2020-02-28 20:19:21.718027400 +0200
@@ -379,7 +379,7 @@
 
         public static const TECHNIQUE_STATISTICS_PERCENT_SIGN:String = "TechniqueStatisticsPercentSign_UI";
 
-        public static const TECHNIQUE_STATISTIC_TAB:String = "TechniqueStatisticTab_UI";
+        public static var TECHNIQUE_STATISTIC_TAB:String = "TechniqueStatisticTab_UI";
 
         public static const TECHNIQUE_ACHIEVEMENT_TAB:String = "TechniqueAchievementTab_UI";
 
diff -Naur lobby.orig/net/wg/gui/components/tooltips/helpers/Utils.as lobby/net/wg/gui/components/tooltips/helpers/Utils.as
--- lobby.orig/net/wg/gui/components/tooltips/helpers/Utils.as	2020-02-28 20:16:00.221922200 +0200
+++ lobby/net/wg/gui/components/tooltips/helpers/Utils.as	2020-02-28 20:19:21.731521400 +0200
@@ -75,7 +75,7 @@
 
         public const STATUS_INFO:String = "info";
 
-        public const allowStatuses:Array = [this.STATUS_ADDITIONAL,this.STATUS_RENTED,this.STATUS_CRITICAL,this.STATUS_WARNING,this.STATUS_INFO];
+        public const allowStatuses:Array = [STATUS_ADDITIONAL,STATUS_RENTED,STATUS_CRITICAL,STATUS_WARNING,STATUS_INFO];
 
         public const MARGIN_AFTER_BLOCK:Number = 15;
 
diff -Naur lobby.orig/net/wg/gui/lobby/battleResults/BattleResults.as lobby/net/wg/gui/lobby/battleResults/BattleResults.as
--- lobby.orig/net/wg/gui/lobby/battleResults/BattleResults.as	2020-02-28 20:16:25.223296100 +0200
+++ lobby/net/wg/gui/lobby/battleResults/BattleResults.as	2020-02-28 20:19:21.748018600 +0200
@@ -41,6 +41,12 @@
 
         private var _emblemLoadingDelegates:Dictionary;
 
+        // XFW
+        public function get xfw_data():BattleResultsVO
+        {
+            return _data;
+        }
+
         private var _data:BattleResultsVO = null;
 
         public function BattleResults()
diff -Naur lobby.orig/net/wg/gui/lobby/battleResults/CommonStats.as lobby/net/wg/gui/lobby/battleResults/CommonStats.as
--- lobby.orig/net/wg/gui/lobby/battleResults/CommonStats.as	2020-02-28 20:16:24.083238000 +0200
+++ lobby/net/wg/gui/lobby/battleResults/CommonStats.as	2020-02-28 20:19:21.765514900 +0200
@@ -145,6 +145,12 @@
 
         private var _progressReport:SubtasksList;
 
+        // XFW
+        public function get xfw_progressReport():SubtasksList
+        {
+            return _progressReport;
+        }
+
         private var _creditsCounterNumber:Number;
 
         private var _xpCounterNumber:Number;
diff -Naur lobby.orig/net/wg/gui/lobby/battleResults/components/BattleResultImageSwitcherView.as lobby/net/wg/gui/lobby/battleResults/components/BattleResultImageSwitcherView.as
--- lobby.orig/net/wg/gui/lobby/battleResults/components/BattleResultImageSwitcherView.as	2020-02-28 20:16:10.394615900 +0200
+++ lobby/net/wg/gui/lobby/battleResults/components/BattleResultImageSwitcherView.as	2020-02-28 20:26:01.135929600 +0200
@@ -31,9 +31,9 @@
         public function setVehicleIcon(param1:Boolean, param2:String, param3:int) : void
         {
             var _loc4_:VehicleIconPreset = null;
-            if(this.vehicleIcon.visible = param1)
+            if(this.vehicleIcon.visible == param1)
             {
-                _loc4_ = BattleResultImageSwitcherView.getPreset(param3);
+                //_loc4_ = BattleResultImageSwitcherView.getPreset(param3);
                 App.utils.asserter.assertNotNull(_loc4_,"Icon preset for level: " + param3 + Errors.WASNT_FOUND);
                 this.vehicleIcon.source = RES_SHOP.getVehicleIcon(_loc4_.size,param2);
                 this.vehicleIcon.scaleX = this.vehicleIcon.scaleY = _loc4_.scale;
diff -Naur lobby.orig/net/wg/gui/lobby/battleResults/data/PersonalDataVO.as lobby/net/wg/gui/lobby/battleResults/data/PersonalDataVO.as
--- lobby.orig/net/wg/gui/lobby/battleResults/data/PersonalDataVO.as	2020-02-28 20:16:26.889514400 +0200
+++ lobby/net/wg/gui/lobby/battleResults/data/PersonalDataVO.as	2020-02-28 20:19:21.798600400 +0200
@@ -396,7 +396,7 @@
             if(param1)
             {
                 this.achievementsRight = new DataProvider();
-                fillAchievements(this.achievementsRight,param1);
+                fillAchievements(this.achievementsRight as Array,param1);
             }
         }
 
@@ -405,7 +405,7 @@
             if(param1)
             {
                 this.achievementsLeft = new DataProvider();
-                fillAchievements(this.achievementsLeft,param1);
+                fillAchievements(this.achievementsLeft as Array,param1);
             }
         }
     }
diff -Naur lobby.orig/net/wg/gui/lobby/components/SmallSkillsList.as lobby/net/wg/gui/lobby/components/SmallSkillsList.as
--- lobby.orig/net/wg/gui/lobby/components/SmallSkillsList.as	2020-02-28 20:16:31.094759400 +0200
+++ lobby/net/wg/gui/lobby/components/SmallSkillsList.as	2020-02-28 20:19:21.814653100 +0200
@@ -14,7 +14,7 @@
 
         private static const PERCENT_SIGN:String = "%";
 
-        private static const MAX_RENDER_SKILLS:int = 5;
+        public static var MAX_RENDER_SKILLS:int = 5;
 
         public var skills:TileList = null;
 
diff -Naur lobby.orig/net/wg/gui/lobby/hangar/Hangar.as lobby/net/wg/gui/lobby/hangar/Hangar.as
--- lobby.orig/net/wg/gui/lobby/hangar/Hangar.as	2020-02-28 20:16:11.425977000 +0200
+++ lobby/net/wg/gui/lobby/hangar/Hangar.as	2020-02-28 20:19:21.831655200 +0200
@@ -142,6 +142,11 @@
 
         public var dqWidget:DailyQuestWidget;
 
+        public function get xfw_header():HangarHeader
+        {
+            return _header;
+        }
+
         private var _header:HangarHeader;
 
         private var _carousel:TankCarousel;
diff -Naur lobby.orig/net/wg/gui/lobby/hangar/ammunitionPanel/AmmunitionPanel.as lobby/net/wg/gui/lobby/hangar/ammunitionPanel/AmmunitionPanel.as
--- lobby.orig/net/wg/gui/lobby/hangar/ammunitionPanel/AmmunitionPanel.as	2020-02-28 20:16:18.177148500 +0200
+++ lobby/net/wg/gui/lobby/hangar/ammunitionPanel/AmmunitionPanel.as	2020-02-28 20:19:21.853659000 +0200
@@ -1050,12 +1050,14 @@
 
         private function animationCompleteHandler(param1:Event) : void
         {
+            /*
             var _loc2_:int = this._animations.indexOf(param1.currentTarget);
             if(_loc2_ != -1)
             {
                 this.disposeAnimation(this._animations[_loc2_]);
                 this._animations.splice(_loc2_,1);
             }
+            */
         }
 
         private function animationLoadedHandler(param1:IconLoaderEvent) : void
diff -Naur lobby.orig/net/wg/gui/lobby/header/LobbyHeader.as lobby/net/wg/gui/lobby/header/LobbyHeader.as
--- lobby.orig/net/wg/gui/lobby/header/LobbyHeader.as	2020-02-28 20:16:13.938423000 +0200
+++ lobby/net/wg/gui/lobby/header/LobbyHeader.as	2020-02-28 20:19:21.875658700 +0200
@@ -100,6 +100,12 @@
 
         private var _headerButtonsHelper:HeaderButtonsHelper;
 
+        // XFW
+        public function get xfw_headerButtonsHelper():HeaderButtonsHelper
+        {
+            return _headerButtonsHelper;
+        }
+
         private var _bubbleTooltip:ToolTipComplex;
 
         private var _actualEnabledVal:Boolean;
diff -Naur lobby.orig/net/wg/gui/lobby/header/headerButtonBar/HeaderButtonContentItem.as lobby/net/wg/gui/lobby/header/headerButtonBar/HeaderButtonContentItem.as
--- lobby.orig/net/wg/gui/lobby/header/headerButtonBar/HeaderButtonContentItem.as	2020-02-28 20:16:08.282924900 +0200
+++ lobby/net/wg/gui/lobby/header/headerButtonBar/HeaderButtonContentItem.as	2020-02-28 20:19:21.895951000 +0200
@@ -323,11 +323,21 @@
             this._needUpdateFontSize = param1;
         }
 
+        public function get xfw_leftPadding():Number
+        {
+            return leftPadding;
+        }
+
         protected function get leftPadding() : Number
         {
             return this._minScreenPadding.left + this._additionalScreenPadding.left * this._wideScreenPrc ^ 0;
         }
 
+        public function get xfw_rightPadding():Number
+        {
+            return rightPadding;
+        }
+
         protected function get rightPadding() : Number
         {
             return this._minScreenPadding.right + this._additionalScreenPadding.right * this._wideScreenPrc ^ 0;
diff -Naur lobby.orig/net/wg/gui/lobby/header/headerButtonBar/HeaderButtonsHelper.as lobby/net/wg/gui/lobby/header/headerButtonBar/HeaderButtonsHelper.as
--- lobby.orig/net/wg/gui/lobby/header/headerButtonBar/HeaderButtonsHelper.as	2020-02-28 20:16:15.634533700 +0200
+++ lobby/net/wg/gui/lobby/header/headerButtonBar/HeaderButtonsHelper.as	2020-02-28 20:19:21.915445700 +0200
@@ -271,6 +271,11 @@
             return this._btnsMap[param1];
         }
 
+        public function xfw_searchButtonById(id:String):HeaderButton
+        {
+            return searchButtonById(id);
+        }
+
         protected function searchButtonById(param1:String) : HeaderButton
         {
             var _loc2_:HeaderButtonVo = this.getButtonDataById(param1);
diff -Naur lobby.orig/net/wg/gui/lobby/profile/ProfileTabNavigator.as lobby/net/wg/gui/lobby/profile/ProfileTabNavigator.as
--- lobby.orig/net/wg/gui/lobby/profile/ProfileTabNavigator.as	2020-02-28 20:16:30.198791300 +0200
+++ lobby/net/wg/gui/lobby/profile/ProfileTabNavigator.as	2020-02-28 20:19:21.933434500 +0200
@@ -40,8 +40,24 @@
 
         private var _data:ProfileMenuInfoVO = null;
 
+        // XFW
+        public function get xfw_data():ProfileMenuInfoVO
+        {
+            return _data;
+        }
+
         private var _sectionsDataUtil:LinkageUtils;
 
+        // XFW
+        public function get xfw_sectionsDataUtil():LinkageUtils
+        {
+            return _sectionsDataUtil;
+        }
+        public function set xfw_sectionsDataUtil(value:LinkageUtils):void
+        {
+            _sectionsDataUtil = value;
+        }
+
         private var _centerOffset:int = 0;
 
         private var _countersToSet:Vector.<CountersVo> = null;
diff -Naur lobby.orig/net/wg/gui/lobby/profile/pages/technique/TechniqueStatisticTab.as lobby/net/wg/gui/lobby/profile/pages/technique/TechniqueStatisticTab.as
--- lobby.orig/net/wg/gui/lobby/profile/pages/technique/TechniqueStatisticTab.as	2020-02-28 20:16:29.080758300 +0200
+++ lobby/net/wg/gui/lobby/profile/pages/technique/TechniqueStatisticTab.as	2020-02-28 20:19:21.951422900 +0200
@@ -40,6 +40,12 @@
 
         private var _group:TechDetailedUnitGroup = null;
 
+        // XFW
+        public function get xfw_group():TechDetailedUnitGroup
+        {
+            return _group;
+        }
+
         public function TechniqueStatisticTab()
         {
             this._tweenManager = new ExcludeTweenManager();
diff -Naur lobby.orig/net/wg/gui/lobby/profile/pages/technique/data/TechniqueListVehicleVO.as lobby/net/wg/gui/lobby/profile/pages/technique/data/TechniqueListVehicleVO.as
--- lobby.orig/net/wg/gui/lobby/profile/pages/technique/data/TechniqueListVehicleVO.as	2020-02-28 20:16:16.444063900 +0200
+++ lobby/net/wg/gui/lobby/profile/pages/technique/data/TechniqueListVehicleVO.as	2020-02-28 20:19:21.965430800 +0200
@@ -4,6 +4,9 @@
 
     public class TechniqueListVehicleVO extends DAAPIDataClass
     {
+        // XVM
+        public var xvm_xte:int;
+        public var xvm_xte_flag:int;
 
         public var id:int;
 
diff -Naur lobby.orig/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as lobby/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as
--- lobby.orig/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as	2020-02-28 20:16:25.178308300 +0200
+++ lobby/net/wg/gui/lobby/techtree/math/ADG_ItemLevelsBuilder.as	2020-02-28 20:19:21.978423400 +0200
@@ -530,6 +530,7 @@
             var _loc16_:* = NaN;
             var _loc17_:* = NaN;
             var _loc3_:Array = [];
+            /*
             if(param1 == 0)
             {
                 return _loc3_;
@@ -575,6 +576,7 @@
                 }
                 _loc15_++;
             }
+            */
             return _loc3_;
         }
 
diff -Naur lobby.orig/net/wg/gui/messenger/ChannelComponent.as lobby/net/wg/gui/messenger/ChannelComponent.as
--- lobby.orig/net/wg/gui/messenger/ChannelComponent.as	2020-02-28 20:16:15.611548500 +0200
+++ lobby/net/wg/gui/messenger/ChannelComponent.as	2020-02-28 20:19:21.998388500 +0200
@@ -30,7 +30,7 @@
 
         public var messageAreaScrollBar:ScrollBar = null;
 
-        public var messageArea:TextAreaSimple = null;
+        public var messageArea:net.wg.gui.components.advanced.TextAreaSimple = null;
 
         public var messageInput:TextInput = null;
 
diff -Naur lobby.orig/net/wg/gui/messenger/ContactsListPopover.as lobby/net/wg/gui/messenger/ContactsListPopover.as
--- lobby.orig/net/wg/gui/messenger/ContactsListPopover.as	2020-02-28 20:16:13.547729000 +0200
+++ lobby/net/wg/gui/messenger/ContactsListPopover.as	2020-02-28 20:19:22.016406100 +0200
@@ -56,6 +56,12 @@
 
         private var _linkageUtils:LinkageUtils;
 
+        // XFW
+        public function get xfw_linkageUtils():LinkageUtils
+        {
+            return _linkageUtils;
+        }
+
         private var _initData:ContactsWindowInitVO = null;
 
         private var _focusChain:Vector.<InteractiveObject> = null;
diff -Naur lobby.orig/net/wg/gui/messenger/controls/ContactsTreeItemRenderer.as lobby/net/wg/gui/messenger/controls/ContactsTreeItemRenderer.as
--- lobby.orig/net/wg/gui/messenger/controls/ContactsTreeItemRenderer.as	2020-02-28 20:16:09.595114300 +0200
+++ lobby/net/wg/gui/messenger/controls/ContactsTreeItemRenderer.as	2020-02-28 20:19:22.033404000 +0200
@@ -29,12 +29,32 @@
 
         private var _dashedArea:EmptyHighlightArea;
 
+        // XFW
+        public function get xfw_contactItem():ContactItem
+        {
+            return _contactItem;
+        }
+        public function set xfw_contactItem(value:ContactItem):void
+        {
+            _contactItem = value;
+        }
+
         private var _contactItem:ContactItem;
 
         private var _mainGroupItem:MainGroupItem;
 
         private var _contactGroupItem:ContactGroupItem;
 
+        // XFW
+        public function get xfw_currentContentItem():UIComponent
+        {
+            return _currentContentItem;
+        }
+        public function set xfw_currentContentItem(value:UIComponent):void
+        {
+            _currentContentItem = value;
+        }
+
         private var _currentContentItem:UIComponent;
 
         private var _isNewDataIsEqualOldData:Boolean = true;
diff -Naur lobby.orig/net/wg/gui/messenger/data/ContactItemVO.as lobby/net/wg/gui/messenger/data/ContactItemVO.as
--- lobby.orig/net/wg/gui/messenger/data/ContactItemVO.as	2020-02-28 20:16:18.282341200 +0200
+++ lobby/net/wg/gui/messenger/data/ContactItemVO.as	2020-02-28 20:19:22.050394900 +0200
@@ -5,6 +5,7 @@
 
     public class ContactItemVO extends DAAPIDataClass
     {
+        public var xvm_contact_data:Object = null;
 
         public static const USER_PROPS:String = "userProps";
 
diff -Naur lobby.orig/net/wg/gui/messenger/meta/impl/ContactNoteManageViewMeta.as lobby/net/wg/gui/messenger/meta/impl/ContactNoteManageViewMeta.as
--- lobby.orig/net/wg/gui/messenger/meta/impl/ContactNoteManageViewMeta.as	2020-02-28 20:16:12.847666200 +0200
+++ lobby/net/wg/gui/messenger/meta/impl/ContactNoteManageViewMeta.as	2020-02-28 20:19:22.064385600 +0200
@@ -33,7 +33,7 @@
             this.sendData(param1);
         }
 
-        public final function as_setUserProps(param1:Object) : void
+        public function as_setUserProps(param1:Object) : void
         {
             var _loc2_:ContactUserPropVO = this._contactUserPropVO;
             this._contactUserPropVO = new ContactUserPropVO(param1);
diff -Naur lobby.orig/net/wg/gui/notification/NotificationPopUpViewer.as lobby/net/wg/gui/notification/NotificationPopUpViewer.as
--- lobby.orig/net/wg/gui/notification/NotificationPopUpViewer.as	2020-02-28 20:16:06.900781600 +0200
+++ lobby/net/wg/gui/notification/NotificationPopUpViewer.as	2020-02-28 20:19:22.079375800 +0200
@@ -39,6 +39,12 @@
 
         private var _popupClass:Class = null;
 
+        // XFW
+        public function set xfw_popupClass(value:Class):void
+        {
+            _popupClass = value;
+        }
+
         private var _popupPadding:uint = 0;
 
         private var _isMessagesCountInvalid:Boolean = false;
diff -Naur lobby.orig/net/wg/gui/rally/events/RallyViewsEvent.as lobby/net/wg/gui/rally/events/RallyViewsEvent.as
--- lobby.orig/net/wg/gui/rally/events/RallyViewsEvent.as	2020-02-28 20:16:21.297371900 +0200
+++ lobby/net/wg/gui/rally/events/RallyViewsEvent.as	2020-02-28 20:19:22.098364200 +0200
@@ -35,7 +35,7 @@
 
         public static const VEH_BTN_ROLL_OUT:String = "vehBtnRollOut";
 
-        public var data;
+        public var data:*;
 
         public function RallyViewsEvent(param1:String, param2:* = null)
         {
diff -Naur lobby.orig/net/wg/infrastructure/managers/IContainerManager.as lobby/net/wg/infrastructure/managers/IContainerManager.as
--- lobby.orig/net/wg/infrastructure/managers/IContainerManager.as	2020-02-28 20:15:49.226737200 +0200
+++ lobby/net/wg/infrastructure/managers/IContainerManager.as	2020-02-28 20:19:22.110356200 +0200
@@ -23,6 +23,8 @@
 
         function storeVisibleContainers() : void;
 
+        function get loader() : ILoaderManager;
+
         function set loader(param1:ILoaderManager) : void;
 
         function get lastFocusedView() : IView;
diff -Naur lobby.orig/net/wg/infrastructure/managers/impl/ContainerManagerBase.as lobby/net/wg/infrastructure/managers/impl/ContainerManagerBase.as
--- lobby.orig/net/wg/infrastructure/managers/impl/ContainerManagerBase.as	2020-02-28 20:15:50.221118900 +0200
+++ lobby/net/wg/infrastructure/managers/impl/ContainerManagerBase.as	2020-02-28 20:19:22.115353300 +0200
@@ -651,6 +651,7 @@
 
         private function onStartLoadViewHandler(param1:LoaderEvent) : void
         {
+            /*
             var _loc2_:String = param1.config.type;
             var _loc3_:String = param1.config.alias;
             var _loc4_:Vector.<AliasVO> = this._containersForLoadingViews[_loc2_];
@@ -662,6 +663,7 @@
             {
                 _loc4_.push(new AliasVO(_loc3_,param1.name));
             }
+            */
         }
 
         private function onViewLoadingCancelHandler(param1:LoaderEvent) : void
diff -Naur lobby.orig/scaleform/clik/data/DataProvider.as lobby/scaleform/clik/data/DataProvider.as
--- lobby.orig/scaleform/clik/data/DataProvider.as	2020-02-28 20:15:55.782673000 +0200
+++ lobby/scaleform/clik/data/DataProvider.as	2020-02-28 20:19:22.137962500 +0200
@@ -5,8 +5,9 @@
     import flash.events.EventDispatcher;
     import flash.events.Event;
 
-    public dynamic class DataProvider extends Array implements IDataProvider, IEventDispatcher
+    public dynamic class DataProvider extends Object implements IDataProvider, IEventDispatcher
     {
+        public function get length() : uint { return 0; };
 
         protected var dispatcher:EventDispatcher;
 
diff -Naur lobby.orig/scaleform/clik/ui/InputDetails.as lobby/scaleform/clik/ui/InputDetails.as
--- lobby.orig/scaleform/clik/ui/InputDetails.as	2020-02-28 20:15:52.497709500 +0200
+++ lobby/scaleform/clik/ui/InputDetails.as	2020-02-28 20:19:22.142336600 +0200
@@ -7,7 +7,7 @@
 
         public var code:Number;
 
-        public var value;
+        public var value:*;
 
         public var navEquivalent:String;
 
