diff -Naur Application.orig-0.orig/net/wg/gui/components/controls/Carousel.class.asasm Application.orig-0/net/wg/gui/components/controls/Carousel.class.asasm
--- Application.orig-0.orig/net/wg/gui/components/controls/Carousel.class.asasm	2014-08-23 23:23:07.228183400 +0300
+++ Application.orig-0/net/wg/gui/components/controls/Carousel.class.asasm	2014-08-24 00:46:42.930971600 +0300
@@ -63,7 +63,7 @@
   trait slot QName(ProtectedNamespace("net.wg.gui.components.controls:Carousel"), "lastDx") type QName(PackageNamespace(""), "Number") value Integer(0) end
   trait slot QName(ProtectedNamespace("net.wg.gui.components.controls:Carousel"), "allowDrag") type QName(PackageNamespace(""), "Boolean") value False() end
   trait slot QName(ProtectedNamespace("net.wg.gui.components.controls:Carousel"), "selectedItemRenderer") type QName(PackageNamespace("scaleform.clik.interfaces"), "IListItemRenderer") value Null() end
-  trait slot QName(PrivateNamespace(null, "net.wg.gui.components.controls:Carousel/instance#0"), "scopeWidth") type QName(PackageNamespace(""), "Number") value Integer(0) end
+  trait slot QName(ProtectedNamespace("net.wg.gui.components.controls:Carousel"), "scopeWidth") type QName(PackageNamespace(""), "Number") value Integer(0) end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.components.controls:Carousel/instance#0"), "courseFactor") type QName(PackageNamespace(""), "Number") value Integer(0) end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.components.controls:Carousel/instance#0"), "slideAcceleratorAfterDrag") type QName(PackageNamespace(""), "Number") value Integer(16) end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.components.controls:Carousel/instance#0"), "slideAcceleratorByArrow") type QName(PackageNamespace(""), "Number") value Integer(7) end
@@ -1384,7 +1384,7 @@
       getlocal0
       getproperty         QName(ProtectedNamespace("net.wg.gui.components.controls:Carousel"), "_defContainerPos")
       getlocal0
-      getproperty         QName(PrivateNamespace(null, "net.wg.gui.components.controls:Carousel/instance#0"), "scopeWidth")
+      getproperty         QName(ProtectedNamespace("net.wg.gui.components.controls:Carousel"), "scopeWidth")
       getlocal0
       getproperty         QName(PackageNamespace(""), "renderersMask")
       getproperty         QName(PackageNamespace(""), "width")
@@ -1455,7 +1455,7 @@
       getlocal0
       getproperty         QName(ProtectedNamespace("net.wg.gui.components.controls:Carousel"), "_defContainerPos")
       getlocal0
-      getproperty         QName(PrivateNamespace(null, "net.wg.gui.components.controls:Carousel/instance#0"), "scopeWidth")
+      getproperty         QName(ProtectedNamespace("net.wg.gui.components.controls:Carousel"), "scopeWidth")
       getlocal0
       getproperty         QName(PackageNamespace(""), "renderersMask")
       getproperty         QName(PackageNamespace(""), "width")
@@ -1590,7 +1590,7 @@
       pushbyte            2
       multiply
       add
-      initproperty        QName(PrivateNamespace(null, "net.wg.gui.components.controls:Carousel/instance#0"), "scopeWidth")
+      initproperty        QName(ProtectedNamespace("net.wg.gui.components.controls:Carousel"), "scopeWidth")
 
       returnvoid
      end ; code
@@ -3164,7 +3164,7 @@
       iffalse             L38
 
       getlocal0
-      getproperty         QName(PrivateNamespace(null, "net.wg.gui.components.controls:Carousel/instance#0"), "scopeWidth")
+      getproperty         QName(ProtectedNamespace("net.wg.gui.components.controls:Carousel"), "scopeWidth")
       getlocal0
       getproperty         QName(PackageNamespace(""), "renderersMask")
       getproperty         QName(PackageNamespace(""), "width")
