diff -Naur Application.orig-0.orig/net/wg/gui/lobby/header/LobbyHeader.class.asasm Application.orig-0/net/wg/gui/lobby/header/LobbyHeader.class.asasm
--- Application.orig-0.orig/net/wg/gui/lobby/header/LobbyHeader.class.asasm	2015-01-08 15:57:51.861988900 +0200
+++ Application.orig-0/net/wg/gui/lobby/header/LobbyHeader.class.asasm	2015-01-08 16:01:21.902283000 +0200
@@ -41,6 +41,26 @@
   trait slot QName(PackageNamespace(""), "headerButtonBar") type QName(PackageNamespace("net.wg.gui.lobby.header.headerButtonBar"), "HeaderButtonBar") end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.header:LobbyHeader"), "_mainMenuHelper") type QName(PackageNamespace("net.wg.gui.lobby.header.mainMenuButtonBar"), "MainMenuHelper") end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.header:LobbyHeader"), "_headerButtonsHelper") type QName(PackageNamespace("net.wg.gui.lobby.header.headerButtonBar"), "HeaderButtonsHelper") end
+  trait getter QName(PackageNamespace(""), "xvm_headerButtonsHelper")
+   method
+    refid "net.wg.gui.lobby.header:LobbyHeader/xvm_headerButtonsHelper/getter"
+    returns QName(PackageNamespace("net.wg.gui.lobby.header.headerButtonBar"), "HeaderButtonsHelper")
+    body
+     maxstack 1
+     localcount 1
+     initscopedepth 5
+     maxscopedepth 6
+     code
+      getlocal0
+      pushscope
+
+      getlocal0
+      getproperty         QName(PrivateNamespace(null, "net.wg.gui.lobby.header:LobbyHeader"), "_headerButtonsHelper")
+      returnvalue
+     end ; code
+    end ; body
+   end ; method
+  end ; trait
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.header:LobbyHeader"), "_isShowHelpLayout") type QName(PackageNamespace(""), "Boolean") value False() end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.header:LobbyHeader"), "_bubbleTooltip") type QName(PackageNamespace("net.wg.gui.components.tooltips"), "ToolTipComplex") end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.header:LobbyHeader"), "_currentScreen") type QName(PackageNamespace(""), "String") value Utf8("") end
diff -Naur Application.orig-0.orig/net/wg/gui/lobby/profile/ProfileTabNavigator.class.asasm Application.orig-0/net/wg/gui/lobby/profile/ProfileTabNavigator.class.asasm
--- Application.orig-0.orig/net/wg/gui/lobby/profile/ProfileTabNavigator.class.asasm	2015-01-08 15:57:49.071985000 +0200
+++ Application.orig-0/net/wg/gui/lobby/profile/ProfileTabNavigator.class.asasm	2015-01-08 16:02:50.862407500 +0200
@@ -81,6 +81,26 @@
   trait slot QName(PackageNamespace(""), "bar") type QName(PackageNamespace("net.wg.gui.components.advanced"), "ButtonBarEx") end
   trait slot QName(PackageNamespace(""), "viewStack") type QName(PackageNamespace("net.wg.gui.lobby.profile.components"), "ResizableViewStack") end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.profile:ProfileTabNavigator"), "initData") type QName(PackageNamespace("net.wg.gui.lobby.profile"), "ProfileMenuInfoVO") end
+  trait getter QName(PackageNamespace(""), "xvm_initData")
+   method
+    refid "net.wg.gui.lobby.profile:ProfileTabNavigator/xvm_initData/getter"
+    returns QName(PackageNamespace("net.wg.gui.lobby.profile"), "ProfileMenuInfoVO")
+    body
+     maxstack 1
+     localcount 1
+     initscopedepth 11
+     maxscopedepth 12
+     code
+      getlocal0
+      pushscope
+
+      getlocal0
+      getproperty         QName(PrivateNamespace(null, "net.wg.gui.lobby.profile:ProfileTabNavigator"), "initData")
+      returnvalue
+     end ; code
+    end ; body
+   end ; method
+  end ; trait
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.profile:ProfileTabNavigator"), "_sectionsDataUtil") type QName(PackageNamespace("net.wg.gui.lobby.profile"), "LinkageUtils") end
   trait slot QName(PrivateNamespace(null, "net.wg.gui.lobby.profile:ProfileTabNavigator"), "_centerOffset") type QName(PackageNamespace(""), "int") value Integer(0) end
   trait method QName(ProtectedNamespace("net.wg.gui.lobby.profile:ProfileTabNavigator"), "configUI") flag OVERRIDE
