diff -Naur Application.orig-0.orig/net/wg/gui/messenger/windows/ContactsWindow.class.asasm Application.orig-0/net/wg/gui/messenger/windows/ContactsWindow.class.asasm
--- Application.orig-0.orig/net/wg/gui/messenger/windows/ContactsWindow.class.asasm	2014-10-04 01:18:26.853063700 +0300
+++ Application.orig-0/net/wg/gui/messenger/windows/ContactsWindow.class.asasm	2014-10-04 01:23:57.102698200 +0300
@@ -824,9 +824,14 @@
       setlocal3
 
       getlocal2
+      pushstring          "xvm.comments.UI::UI_ContactsListForm"
+      ifeq                XVM1
+
+      getlocal2
       getlex              QName(PrivateNamespace(null, "net.wg.gui.messenger.windows:ContactsWindow"), "VIEW_LIST")
       ifne                L45
 
+XVM1:
       pushstring          "friendsDP"
       getlocal0
       getproperty         QName(ProtectedNamespace("net.wg.gui.messenger.windows:ContactsWindow"), "friendsDP")
